{"cells":[{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":69162,"status":"ok","timestamp":1763222657432,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"iV9oZ2IWQN9F","outputId":"81303339-8bcd-4264-e7fa-38e8b1bcff09"},"outputs":[{"name":"stdout","output_type":"stream","text":["Mounted at /content/drive\n"]}],"source":["from google.colab import drive\n","drive.mount('/content/drive', force_remount=True)\n"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":108,"status":"ok","timestamp":1763223181244,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"1MDYwz6eSlyG","outputId":"b940bb80-9c54-46b0-8b6e-15535340c4ed"},"outputs":[{"name":"stdout","output_type":"stream","text":["Project directory created.\n"]}],"source":["!mkdir -p /content/CTC360/{fastq,trimmed,qc,bam,variants,figures,ref}\n","project_dir = \"/content/CTC360\"\n","print(\"Project directory created.\")\n"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"collapsed":true,"executionInfo":{"elapsed":31333,"status":"ok","timestamp":1763223241844,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"UAPrDtbJSqGZ","outputId":"51ffcaf3-e56d-46d9-c68e-568c65d174e7"},"outputs":[{"name":"stdout","output_type":"stream","text":["Get:1 https://cloud.r-project.org/bin/linux/ubuntu jammy-cran40/ InRelease [3,632 B]\n","Get:2 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64  InRelease [1,581 B]\n","Get:3 https://cli.github.com/packages stable InRelease [3,917 B]\n","Hit:4 http://archive.ubuntu.com/ubuntu jammy InRelease\n","Get:5 http://security.ubuntu.com/ubuntu jammy-security InRelease [129 kB]\n","Get:6 https://r2u.stat.illinois.edu/ubuntu jammy InRelease [6,555 B]\n","Get:7 http://archive.ubuntu.com/ubuntu jammy-updates InRelease [128 kB]\n","Hit:8 https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy InRelease\n","Get:9 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64  Packages [2,143 kB]\n","Hit:10 https://ppa.launchpadcontent.net/graphics-drivers/ppa/ubuntu jammy InRelease\n","Get:11 https://cli.github.com/packages stable/main amd64 Packages [343 B]\n","Hit:12 https://ppa.launchpadcontent.net/ubuntugis/ppa/ubuntu jammy InRelease\n","Get:13 http://archive.ubuntu.com/ubuntu jammy-backports InRelease [127 kB]\n","Get:14 https://r2u.stat.illinois.edu/ubuntu jammy/main all Packages [9,452 kB]\n","Get:15 https://r2u.stat.illinois.edu/ubuntu jammy/main amd64 Packages [2,827 kB]\n","Get:16 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 Packages [1,595 kB]\n","Get:17 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 Packages [3,861 kB]\n","Get:18 http://archive.ubuntu.com/ubuntu jammy-updates/restricted amd64 Packages [6,181 kB]\n","Fetched 26.5 MB in 3s (8,093 kB/s)\n","Reading package lists... Done\n","W: Skipping acquire of configured file 'main/source/Sources' as repository 'https://r2u.stat.illinois.edu/ubuntu jammy InRelease' does not seem to provide it (sources.list entry misspelt?)\n","Reading package lists... Done\n","Building dependency tree... Done\n","Reading state information... Done\n","The following additional packages will be installed:\n","  at-spi2-core blends-common default-jre default-jre-headless\n","  fonts-dejavu-core fonts-dejavu-extra gsettings-desktop-schemas\n","  libapache-pom-java libargs4j-java libatk-bridge2.0-0 libatk-wrapper-java\n","  libatk-wrapper-java-jni libatk1.0-0 libatk1.0-data libatspi2.0-0\n","  libcommons-compress-java libcommons-io-java libcommons-jexl2-java\n","  libcommons-lang3-java libcommons-logging-java libcommons-math3-java\n","  libcommons-parent-java libfindbin-libs-perl libhts3 libhtscodecs2\n","  libhtsjdk-java libjbzip2-java libjson-simple-java libkdf5-2 libncbi-vdb2\n","  libncbi-wvdb2 libngs-java libngs-sdk-dev libngs-sdk2 libsis-base-java\n","  libsis-base-jni libsis-jhdf5-java libsis-jhdf5-jni libsnappy-java\n","  libsnappy-jni libxcomposite1 libxtst6 libxxf86dga1 med-config menu\n","  openjdk-11-jre openjdk-11-jre-headless session-migration x11-utils\n","Suggested packages:\n","  blends-doc libxz-java libcommons-io-java-doc libcommons-lang3-java-doc\n","  libavalon-framework-java libcommons-logging-java-doc\n","  libexcalibur-logkit-java liblog4j1.2-java picard-tools libjson-simple-doc\n","  menu-l10n gksu | kde-runtime | ktsuss libnss-mdns fonts-ipafont-gothic\n","  fonts-ipafont-mincho fonts-wqy-microhei | fonts-wqy-zenhei fonts-indic\n","  cwltool mesa-utils\n","The following NEW packages will be installed:\n","  at-spi2-core blends-common bwa default-jre default-jre-headless fastp fastqc\n","  fonts-dejavu-core fonts-dejavu-extra gsettings-desktop-schemas\n","  libapache-pom-java libargs4j-java libatk-bridge2.0-0 libatk-wrapper-java\n","  libatk-wrapper-java-jni libatk1.0-0 libatk1.0-data libatspi2.0-0\n","  libcommons-compress-java libcommons-io-java libcommons-jexl2-java\n","  libcommons-lang3-java libcommons-logging-java libcommons-math3-java\n","  libcommons-parent-java libfindbin-libs-perl libhts3 libhtscodecs2\n","  libhtsjdk-java libjbzip2-java libjson-simple-java libkdf5-2 libncbi-vdb2\n","  libncbi-wvdb2 libngs-java libngs-sdk-dev libngs-sdk2 libsis-base-java\n","  libsis-base-jni libsis-jhdf5-java libsis-jhdf5-jni libsnappy-java\n","  libsnappy-jni libxcomposite1 libxtst6 libxxf86dga1 med-config menu\n","  openjdk-11-jre openjdk-11-jre-headless samtools session-migration\n","  sra-toolkit x11-utils\n","0 upgraded, 54 newly installed, 0 to remove and 48 not upgraded.\n","Need to get 64.2 MB of archives.\n","After this operation, 229 MB of additional disk space will be used.\n","Get:1 http://archive.ubuntu.com/ubuntu jammy/main amd64 libatspi2.0-0 amd64 2.44.0-3 [80.9 kB]\n","Get:2 http://archive.ubuntu.com/ubuntu jammy/main amd64 libxtst6 amd64 2:1.2.3-1build4 [13.4 kB]\n","Get:3 http://archive.ubuntu.com/ubuntu jammy/main amd64 session-migration amd64 0.3.6 [9,774 B]\n","Get:4 http://archive.ubuntu.com/ubuntu jammy/main amd64 gsettings-desktop-schemas all 42.0-1ubuntu1 [31.1 kB]\n","Get:5 http://archive.ubuntu.com/ubuntu jammy/main amd64 at-spi2-core amd64 2.44.0-3 [54.4 kB]\n","Get:6 http://archive.ubuntu.com/ubuntu jammy/universe amd64 menu amd64 2.1.47ubuntu4 [354 kB]\n","Get:7 http://archive.ubuntu.com/ubuntu jammy/universe amd64 blends-common all 0.7.4ubuntu1 [15.9 kB]\n","Get:8 http://archive.ubuntu.com/ubuntu jammy/universe amd64 bwa amd64 0.7.17-6 [195 kB]\n","Get:9 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 openjdk-11-jre-headless amd64 11.0.28+6-1ubuntu1~22.04.1 [42.6 MB]\n","Get:10 http://archive.ubuntu.com/ubuntu jammy/main amd64 default-jre-headless amd64 2:1.11-72build2 [3,042 B]\n","Get:11 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 openjdk-11-jre amd64 11.0.28+6-1ubuntu1~22.04.1 [214 kB]\n","Get:12 http://archive.ubuntu.com/ubuntu jammy/main amd64 default-jre amd64 2:1.11-72build2 [896 B]\n","Get:13 http://archive.ubuntu.com/ubuntu jammy/universe amd64 fastp amd64 0.20.1+dfsg-1 [193 kB]\n","Get:14 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libcommons-math3-java all 3.6.1-3 [2,016 kB]\n","Get:15 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libjbzip2-java all 0.9.1-7 [42.8 kB]\n","Get:16 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libapache-pom-java all 18-1 [4,720 B]\n","Get:17 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libcommons-parent-java all 43-1 [10.8 kB]\n","Get:18 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libcommons-compress-java all 1.21-1 [575 kB]\n","Get:19 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libcommons-logging-java all 1.2-2 [60.3 kB]\n","Get:20 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libcommons-jexl2-java all 2.1.1-5 [251 kB]\n","Get:21 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libjson-simple-java all 2.3.0-1 [61.0 kB]\n","Get:22 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libngs-sdk2 amd64 2.11.2-3 [65.6 kB]\n","Get:23 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libngs-sdk-dev amd64 2.11.2-3 [111 kB]\n","Get:24 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libngs-java amd64 2.11.2-3 [75.8 kB]\n","Get:25 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libsnappy-jni amd64 1.1.8.3-1 [6,756 B]\n","Get:26 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libsnappy-java all 1.1.8.3-1 [93.9 kB]\n","Get:27 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libhtsjdk-java all 2.24.1+dfsg-1 [1,644 kB]\n","Get:28 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libfindbin-libs-perl all 2.190.02-1 [23.2 kB]\n","Get:29 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libargs4j-java all 2.33-1.1 [137 kB]\n","Get:30 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libcommons-io-java all 2.11.0-2 [297 kB]\n","Get:31 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libcommons-lang3-java all 3.11-1 [526 kB]\n","Get:32 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libsis-base-jni amd64 18.09~pre1+git20180928.45fbd31+dfsg-3 [12.7 kB]\n","Get:33 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libsis-base-java all 18.09~pre1+git20180928.45fbd31+dfsg-3 [96.4 kB]\n","Get:34 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libsis-jhdf5-jni amd64 19.04.0+dfsg-4 [96.6 kB]\n","Get:35 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libsis-jhdf5-java all 19.04.0+dfsg-4 [1,490 kB]\n","Get:36 http://archive.ubuntu.com/ubuntu jammy/universe amd64 fastqc all 0.11.9+dfsg-5 [370 kB]\n","Get:37 http://archive.ubuntu.com/ubuntu jammy/main amd64 fonts-dejavu-core all 2.37-2build1 [1,041 kB]\n","Get:38 http://archive.ubuntu.com/ubuntu jammy/main amd64 fonts-dejavu-extra all 2.37-2build1 [2,041 kB]\n","Get:39 http://archive.ubuntu.com/ubuntu jammy/main amd64 libatk1.0-data all 2.36.0-3build1 [2,824 B]\n","Get:40 http://archive.ubuntu.com/ubuntu jammy/main amd64 libatk1.0-0 amd64 2.36.0-3build1 [51.9 kB]\n","Get:41 http://archive.ubuntu.com/ubuntu jammy/main amd64 libatk-bridge2.0-0 amd64 2.38.0-3 [66.6 kB]\n","Get:42 http://archive.ubuntu.com/ubuntu jammy/main amd64 libxcomposite1 amd64 1:0.4.5-1build2 [7,192 B]\n","Get:43 http://archive.ubuntu.com/ubuntu jammy/main amd64 libxxf86dga1 amd64 2:1.1.5-0ubuntu3 [12.6 kB]\n","Get:44 http://archive.ubuntu.com/ubuntu jammy/main amd64 x11-utils amd64 7.7+5build2 [206 kB]\n","Get:45 http://archive.ubuntu.com/ubuntu jammy/main amd64 libatk-wrapper-java all 0.38.0-5build1 [53.1 kB]\n","Get:46 http://archive.ubuntu.com/ubuntu jammy/main amd64 libatk-wrapper-java-jni amd64 0.38.0-5build1 [49.0 kB]\n","Get:47 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libhtscodecs2 amd64 1.1.1-3 [53.2 kB]\n","Get:48 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libhts3 amd64 1.13+ds-2build1 [390 kB]\n","Get:49 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libkdf5-2 amd64 2.11.2+dfsg-4build2 [14.7 kB]\n","Get:50 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libncbi-vdb2 amd64 2.11.2+dfsg-4build2 [1,364 kB]\n","Get:51 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libncbi-wvdb2 amd64 2.11.2+dfsg-4build2 [1,252 kB]\n","Get:52 http://archive.ubuntu.com/ubuntu jammy/universe amd64 samtools amd64 1.13-4 [520 kB]\n","Get:53 http://archive.ubuntu.com/ubuntu jammy/universe amd64 sra-toolkit amd64 2.11.3+dfsg-1ubuntu1 [5,277 kB]\n","Get:54 http://archive.ubuntu.com/ubuntu jammy/universe amd64 med-config all 3.7.1 [11.6 kB]\n","Fetched 64.2 MB in 3s (22.6 MB/s)\n","Extracting templates from packages: 100%\n","Preconfiguring packages ...\n","Selecting previously unselected package libatspi2.0-0:amd64.\n","(Reading database ... 121703 files and directories currently installed.)\n","Preparing to unpack .../00-libatspi2.0-0_2.44.0-3_amd64.deb ...\n","Unpacking libatspi2.0-0:amd64 (2.44.0-3) ...\n","Selecting previously unselected package libxtst6:amd64.\n","Preparing to unpack .../01-libxtst6_2%3a1.2.3-1build4_amd64.deb ...\n","Unpacking libxtst6:amd64 (2:1.2.3-1build4) ...\n","Selecting previously unselected package session-migration.\n","Preparing to unpack .../02-session-migration_0.3.6_amd64.deb ...\n","Unpacking session-migration (0.3.6) ...\n","Selecting previously unselected package gsettings-desktop-schemas.\n","Preparing to unpack .../03-gsettings-desktop-schemas_42.0-1ubuntu1_all.deb ...\n","Unpacking gsettings-desktop-schemas (42.0-1ubuntu1) ...\n","Selecting previously unselected package at-spi2-core.\n","Preparing to unpack .../04-at-spi2-core_2.44.0-3_amd64.deb ...\n","Unpacking at-spi2-core (2.44.0-3) ...\n","Selecting previously unselected package menu.\n","Preparing to unpack .../05-menu_2.1.47ubuntu4_amd64.deb ...\n","Unpacking menu (2.1.47ubuntu4) ...\n","Selecting previously unselected package blends-common.\n","Preparing to unpack .../06-blends-common_0.7.4ubuntu1_all.deb ...\n","Unpacking blends-common (0.7.4ubuntu1) ...\n","Selecting previously unselected package bwa.\n","Preparing to unpack .../07-bwa_0.7.17-6_amd64.deb ...\n","Unpacking bwa (0.7.17-6) ...\n","Selecting previously unselected package openjdk-11-jre-headless:amd64.\n","Preparing to unpack .../08-openjdk-11-jre-headless_11.0.28+6-1ubuntu1~22.04.1_amd64.deb ...\n","Unpacking openjdk-11-jre-headless:amd64 (11.0.28+6-1ubuntu1~22.04.1) ...\n","Selecting previously unselected package default-jre-headless.\n","Preparing to unpack .../09-default-jre-headless_2%3a1.11-72build2_amd64.deb ...\n","Unpacking default-jre-headless (2:1.11-72build2) ...\n","Selecting previously unselected package openjdk-11-jre:amd64.\n","Preparing to unpack .../10-openjdk-11-jre_11.0.28+6-1ubuntu1~22.04.1_amd64.deb ...\n","Unpacking openjdk-11-jre:amd64 (11.0.28+6-1ubuntu1~22.04.1) ...\n","Selecting previously unselected package default-jre.\n","Preparing to unpack .../11-default-jre_2%3a1.11-72build2_amd64.deb ...\n","Unpacking default-jre (2:1.11-72build2) ...\n","Selecting previously unselected package fastp.\n","Preparing to unpack .../12-fastp_0.20.1+dfsg-1_amd64.deb ...\n","Unpacking fastp (0.20.1+dfsg-1) ...\n","Selecting previously unselected package libcommons-math3-java.\n","Preparing to unpack .../13-libcommons-math3-java_3.6.1-3_all.deb ...\n","Unpacking libcommons-math3-java (3.6.1-3) ...\n","Selecting previously unselected package libjbzip2-java.\n","Preparing to unpack .../14-libjbzip2-java_0.9.1-7_all.deb ...\n","Unpacking libjbzip2-java (0.9.1-7) ...\n","Selecting previously unselected package libapache-pom-java.\n","Preparing to unpack .../15-libapache-pom-java_18-1_all.deb ...\n","Unpacking libapache-pom-java (18-1) ...\n","Selecting previously unselected package libcommons-parent-java.\n","Preparing to unpack .../16-libcommons-parent-java_43-1_all.deb ...\n","Unpacking libcommons-parent-java (43-1) ...\n","Selecting previously unselected package libcommons-compress-java.\n","Preparing to unpack .../17-libcommons-compress-java_1.21-1_all.deb ...\n","Unpacking libcommons-compress-java (1.21-1) ...\n","Selecting previously unselected package libcommons-logging-java.\n","Preparing to unpack .../18-libcommons-logging-java_1.2-2_all.deb ...\n","Unpacking libcommons-logging-java (1.2-2) ...\n","Selecting previously unselected package libcommons-jexl2-java.\n","Preparing to unpack .../19-libcommons-jexl2-java_2.1.1-5_all.deb ...\n","Unpacking libcommons-jexl2-java (2.1.1-5) ...\n","Selecting previously unselected package libjson-simple-java.\n","Preparing to unpack .../20-libjson-simple-java_2.3.0-1_all.deb ...\n","Unpacking libjson-simple-java (2.3.0-1) ...\n","Selecting previously unselected package libngs-sdk2:amd64.\n","Preparing to unpack .../21-libngs-sdk2_2.11.2-3_amd64.deb ...\n","Unpacking libngs-sdk2:amd64 (2.11.2-3) ...\n","Selecting previously unselected package libngs-sdk-dev:amd64.\n","Preparing to unpack .../22-libngs-sdk-dev_2.11.2-3_amd64.deb ...\n","Unpacking libngs-sdk-dev:amd64 (2.11.2-3) ...\n","Selecting previously unselected package libngs-java:amd64.\n","Preparing to unpack .../23-libngs-java_2.11.2-3_amd64.deb ...\n","Unpacking libngs-java:amd64 (2.11.2-3) ...\n","Selecting previously unselected package libsnappy-jni.\n","Preparing to unpack .../24-libsnappy-jni_1.1.8.3-1_amd64.deb ...\n","Unpacking libsnappy-jni (1.1.8.3-1) ...\n","Selecting previously unselected package libsnappy-java.\n","Preparing to unpack .../25-libsnappy-java_1.1.8.3-1_all.deb ...\n","Unpacking libsnappy-java (1.1.8.3-1) ...\n","Selecting previously unselected package libhtsjdk-java.\n","Preparing to unpack .../26-libhtsjdk-java_2.24.1+dfsg-1_all.deb ...\n","Unpacking libhtsjdk-java (2.24.1+dfsg-1) ...\n","Selecting previously unselected package libfindbin-libs-perl.\n","Preparing to unpack .../27-libfindbin-libs-perl_2.190.02-1_all.deb ...\n","Unpacking libfindbin-libs-perl (2.190.02-1) ...\n","Selecting previously unselected package libargs4j-java.\n","Preparing to unpack .../28-libargs4j-java_2.33-1.1_all.deb ...\n","Unpacking libargs4j-java (2.33-1.1) ...\n","Selecting previously unselected package libcommons-io-java.\n","Preparing to unpack .../29-libcommons-io-java_2.11.0-2_all.deb ...\n","Unpacking libcommons-io-java (2.11.0-2) ...\n","Selecting previously unselected package libcommons-lang3-java.\n","Preparing to unpack .../30-libcommons-lang3-java_3.11-1_all.deb ...\n","Unpacking libcommons-lang3-java (3.11-1) ...\n","Selecting previously unselected package libsis-base-jni.\n","Preparing to unpack .../31-libsis-base-jni_18.09~pre1+git20180928.45fbd31+dfsg-3_amd64.deb ...\n","Unpacking libsis-base-jni (18.09~pre1+git20180928.45fbd31+dfsg-3) ...\n","Selecting previously unselected package libsis-base-java.\n","Preparing to unpack .../32-libsis-base-java_18.09~pre1+git20180928.45fbd31+dfsg-3_all.deb ...\n","Unpacking libsis-base-java (18.09~pre1+git20180928.45fbd31+dfsg-3) ...\n","Selecting previously unselected package libsis-jhdf5-jni.\n","Preparing to unpack .../33-libsis-jhdf5-jni_19.04.0+dfsg-4_amd64.deb ...\n","Unpacking libsis-jhdf5-jni (19.04.0+dfsg-4) ...\n","Selecting previously unselected package libsis-jhdf5-java.\n","Preparing to unpack .../34-libsis-jhdf5-java_19.04.0+dfsg-4_all.deb ...\n","Unpacking libsis-jhdf5-java (19.04.0+dfsg-4) ...\n","Selecting previously unselected package fastqc.\n","Preparing to unpack .../35-fastqc_0.11.9+dfsg-5_all.deb ...\n","Unpacking fastqc (0.11.9+dfsg-5) ...\n","Selecting previously unselected package fonts-dejavu-core.\n","Preparing to unpack .../36-fonts-dejavu-core_2.37-2build1_all.deb ...\n","Unpacking fonts-dejavu-core (2.37-2build1) ...\n","Selecting previously unselected package fonts-dejavu-extra.\n","Preparing to unpack .../37-fonts-dejavu-extra_2.37-2build1_all.deb ...\n","Unpacking fonts-dejavu-extra (2.37-2build1) ...\n","Selecting previously unselected package libatk1.0-data.\n","Preparing to unpack .../38-libatk1.0-data_2.36.0-3build1_all.deb ...\n","Unpacking libatk1.0-data (2.36.0-3build1) ...\n","Selecting previously unselected package libatk1.0-0:amd64.\n","Preparing to unpack .../39-libatk1.0-0_2.36.0-3build1_amd64.deb ...\n","Unpacking libatk1.0-0:amd64 (2.36.0-3build1) ...\n","Selecting previously unselected package libatk-bridge2.0-0:amd64.\n","Preparing to unpack .../40-libatk-bridge2.0-0_2.38.0-3_amd64.deb ...\n","Unpacking libatk-bridge2.0-0:amd64 (2.38.0-3) ...\n","Selecting previously unselected package libxcomposite1:amd64.\n","Preparing to unpack .../41-libxcomposite1_1%3a0.4.5-1build2_amd64.deb ...\n","Unpacking libxcomposite1:amd64 (1:0.4.5-1build2) ...\n","Selecting previously unselected package libxxf86dga1:amd64.\n","Preparing to unpack .../42-libxxf86dga1_2%3a1.1.5-0ubuntu3_amd64.deb ...\n","Unpacking libxxf86dga1:amd64 (2:1.1.5-0ubuntu3) ...\n","Selecting previously unselected package x11-utils.\n","Preparing to unpack .../43-x11-utils_7.7+5build2_amd64.deb ...\n","Unpacking x11-utils (7.7+5build2) ...\n","Selecting previously unselected package libatk-wrapper-java.\n","Preparing to unpack .../44-libatk-wrapper-java_0.38.0-5build1_all.deb ...\n","Unpacking libatk-wrapper-java (0.38.0-5build1) ...\n","Selecting previously unselected package libatk-wrapper-java-jni:amd64.\n","Preparing to unpack .../45-libatk-wrapper-java-jni_0.38.0-5build1_amd64.deb ...\n","Unpacking libatk-wrapper-java-jni:amd64 (0.38.0-5build1) ...\n","Selecting previously unselected package libhtscodecs2:amd64.\n","Preparing to unpack .../46-libhtscodecs2_1.1.1-3_amd64.deb ...\n","Unpacking libhtscodecs2:amd64 (1.1.1-3) ...\n","Selecting previously unselected package libhts3:amd64.\n","Preparing to unpack .../47-libhts3_1.13+ds-2build1_amd64.deb ...\n","Unpacking libhts3:amd64 (1.13+ds-2build1) ...\n","Selecting previously unselected package libkdf5-2.\n","Preparing to unpack .../48-libkdf5-2_2.11.2+dfsg-4build2_amd64.deb ...\n","Unpacking libkdf5-2 (2.11.2+dfsg-4build2) ...\n","Selecting previously unselected package libncbi-vdb2.\n","Preparing to unpack .../49-libncbi-vdb2_2.11.2+dfsg-4build2_amd64.deb ...\n","Unpacking libncbi-vdb2 (2.11.2+dfsg-4build2) ...\n","Selecting previously unselected package libncbi-wvdb2.\n","Preparing to unpack .../50-libncbi-wvdb2_2.11.2+dfsg-4build2_amd64.deb ...\n","Unpacking libncbi-wvdb2 (2.11.2+dfsg-4build2) ...\n","Selecting previously unselected package samtools.\n","Preparing to unpack .../51-samtools_1.13-4_amd64.deb ...\n","Unpacking samtools (1.13-4) ...\n","Selecting previously unselected package sra-toolkit.\n","Preparing to unpack .../52-sra-toolkit_2.11.3+dfsg-1ubuntu1_amd64.deb ...\n","Unpacking sra-toolkit (2.11.3+dfsg-1ubuntu1) ...\n","Selecting previously unselected package med-config.\n","Preparing to unpack .../53-med-config_3.7.1_all.deb ...\n","Unpacking med-config (3.7.1) ...\n","Setting up libhtscodecs2:amd64 (1.1.1-3) ...\n","Setting up session-migration (0.3.6) ...\n","Created symlink /etc/systemd/user/graphical-session-pre.target.wants/session-migration.service → /usr/lib/systemd/user/session-migration.service.\n","Setting up libhts3:amd64 (1.13+ds-2build1) ...\n","Setting up bwa (0.7.17-6) ...\n","Setting up libxtst6:amd64 (2:1.2.3-1build4) ...\n","Setting up openjdk-11-jre-headless:amd64 (11.0.28+6-1ubuntu1~22.04.1) ...\n","update-alternatives: using /usr/lib/jvm/java-11-openjdk-amd64/bin/jjs to provide /usr/bin/jjs (jjs) in auto mode\n","update-alternatives: using /usr/lib/jvm/java-11-openjdk-amd64/bin/rmid to provide /usr/bin/rmid (rmid) in auto mode\n","update-alternatives: using /usr/lib/jvm/java-11-openjdk-amd64/bin/pack200 to provide /usr/bin/pack200 (pack200) in auto mode\n","update-alternatives: using /usr/lib/jvm/java-11-openjdk-amd64/bin/unpack200 to provide /usr/bin/unpack200 (unpack200) in auto mode\n","Setting up libngs-sdk2:amd64 (2.11.2-3) ...\n","Setting up libxxf86dga1:amd64 (2:1.1.5-0ubuntu3) ...\n","Setting up openjdk-11-jre:amd64 (11.0.28+6-1ubuntu1~22.04.1) ...\n","Setting up libfindbin-libs-perl (2.190.02-1) ...\n","Setting up libncbi-vdb2 (2.11.2+dfsg-4build2) ...\n","Setting up libatspi2.0-0:amd64 (2.44.0-3) ...\n","Setting up libjson-simple-java (2.3.0-1) ...\n","Setting up libkdf5-2 (2.11.2+dfsg-4build2) ...\n","Setting up samtools (1.13-4) ...\n","Setting up fastp (0.20.1+dfsg-1) ...\n","Setting up libsis-base-jni (18.09~pre1+git20180928.45fbd31+dfsg-3) ...\n","Setting up libncbi-wvdb2 (2.11.2+dfsg-4build2) ...\n","Setting up libapache-pom-java (18-1) ...\n","Setting up fonts-dejavu-core (2.37-2build1) ...\n","Setting up libjbzip2-java (0.9.1-7) ...\n","Setting up libngs-sdk-dev:amd64 (2.11.2-3) ...\n","Setting up libatk1.0-data (2.36.0-3build1) ...\n","Setting up fonts-dejavu-extra (2.37-2build1) ...\n","Setting up libcommons-math3-java (3.6.1-3) ...\n","Setting up libsis-jhdf5-jni (19.04.0+dfsg-4) ...\n","Setting up libatk1.0-0:amd64 (2.36.0-3build1) ...\n","Setting up libngs-java:amd64 (2.11.2-3) ...\n","Setting up menu (2.1.47ubuntu4) ...\n","Setting up libsnappy-jni (1.1.8.3-1) ...\n","Setting up libxcomposite1:amd64 (1:0.4.5-1build2) ...\n","Setting up libargs4j-java (2.33-1.1) ...\n","Setting up blends-common (0.7.4ubuntu1) ...\n","Setting up gsettings-desktop-schemas (42.0-1ubuntu1) ...\n","Setting up sra-toolkit (2.11.3+dfsg-1ubuntu1) ...\n","Setting up med-config (3.7.1) ...\n","Adding group `med' (GID 107) ...\n","Done.\n","Setting up default-jre-headless (2:1.11-72build2) ...\n","Setting up libcommons-parent-java (43-1) ...\n","Setting up libcommons-logging-java (1.2-2) ...\n","Setting up default-jre (2:1.11-72build2) ...\n","Setting up libatk-bridge2.0-0:amd64 (2.38.0-3) ...\n","Setting up libcommons-jexl2-java (2.1.1-5) ...\n","Setting up libcommons-lang3-java (3.11-1) ...\n","Setting up x11-utils (7.7+5build2) ...\n","Setting up libsnappy-java (1.1.8.3-1) ...\n","Setting up libatk-wrapper-java (0.38.0-5build1) ...\n","Setting up libcommons-compress-java (1.21-1) ...\n","Setting up libcommons-io-java (2.11.0-2) ...\n","Setting up libatk-wrapper-java-jni:amd64 (0.38.0-5build1) ...\n","Setting up libsis-base-java (18.09~pre1+git20180928.45fbd31+dfsg-3) ...\n","Setting up libhtsjdk-java (2.24.1+dfsg-1) ...\n","Setting up libsis-jhdf5-java (19.04.0+dfsg-4) ...\n","Setting up fastqc (0.11.9+dfsg-5) ...\n","Processing triggers for mailcap (3.70+nmu1ubuntu1) ...\n","Processing triggers for fontconfig (2.13.1-4.2ubuntu5) ...\n","Processing triggers for hicolor-icon-theme (0.17-2) ...\n","Processing triggers for libglib2.0-0:amd64 (2.72.4-0ubuntu2.6) ...\n","Processing triggers for libc-bin (2.35-0ubuntu3.8) ...\n","/sbin/ldconfig.real: /usr/local/lib/libur_adapter_opencl.so.0 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtbbbind.so.3 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtcm_debug.so.1 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtcm.so.1 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc.so.2 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtbb.so.12 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc_proxy.so.2 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libumf.so.1 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libur_adapter_level_zero_v2.so.0 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libur_loader.so.0 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libur_adapter_level_zero.so.0 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_5.so.3 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_0.so.3 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libhwloc.so.15 is not a symbolic link\n","\n","Processing triggers for man-db (2.10.2-1) ...\n","Setting up at-spi2-core (2.44.0-3) ...\n","Processing triggers for menu (2.1.47ubuntu4) ...\n"]}],"source":["!apt-get update\n","!apt-get install -y sra-toolkit fastqc bwa samtools fastp\n"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"collapsed":true,"executionInfo":{"elapsed":14974,"status":"ok","timestamp":1763223352384,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"6pKKpoJQTOZk","outputId":"337c3e8e-1cf0-4f58-e0e7-cbb6748203e0"},"outputs":[{"name":"stdout","output_type":"stream","text":["--2025-11-15 16:15:36--  https://github.com/broadinstitute/gatk/releases/download/4.4.0.0/gatk-4.4.0.0.zip\n","Resolving github.com (github.com)... 140.82.113.4\n","Connecting to github.com (github.com)|140.82.113.4|:443... connected.\n","HTTP request sent, awaiting response... 302 Found\n","Location: https://release-assets.githubusercontent.com/github-production-release-asset/27452807/0bd42ce9-3dc6-4814-971c-264ebaf66d3d?sp=r&sv=2018-11-09&sr=b&spr=https&se=2025-11-15T17%3A15%3A40Z&rscd=attachment%3B+filename%3Dgatk-4.4.0.0.zip&rsct=application%2Foctet-stream&skoid=96c2d410-5711-43a1-aedd-ab1947aa7ab0&sktid=398a6654-997b-47e9-b12b-9515b896b4de&skt=2025-11-15T16%3A15%3A37Z&ske=2025-11-15T17%3A15%3A40Z&sks=b&skv=2018-11-09&sig=F9fY%2FLEaVBEUvjPzODMwCMesFcBvIMnkzOjV74dhJNs%3D&jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmVsZWFzZS1hc3NldHMuZ2l0aHVidXNlcmNvbnRlbnQuY29tIiwia2V5Ijoia2V5MSIsImV4cCI6MTc2MzIyNjkzNywibmJmIjoxNzYzMjIzMzM3LCJwYXRoIjoicmVsZWFzZWFzc2V0cHJvZHVjdGlvbi5ibG9iLmNvcmUud2luZG93cy5uZXQifQ.KzIcTHG91lu0ambiXqS2gVklpB7nrFOmUT3G-VvKLls&response-content-disposition=attachment%3B%20filename%3Dgatk-4.4.0.0.zip&response-content-type=application%2Foctet-stream [following]\n","--2025-11-15 16:15:37--  https://release-assets.githubusercontent.com/github-production-release-asset/27452807/0bd42ce9-3dc6-4814-971c-264ebaf66d3d?sp=r&sv=2018-11-09&sr=b&spr=https&se=2025-11-15T17%3A15%3A40Z&rscd=attachment%3B+filename%3Dgatk-4.4.0.0.zip&rsct=application%2Foctet-stream&skoid=96c2d410-5711-43a1-aedd-ab1947aa7ab0&sktid=398a6654-997b-47e9-b12b-9515b896b4de&skt=2025-11-15T16%3A15%3A37Z&ske=2025-11-15T17%3A15%3A40Z&sks=b&skv=2018-11-09&sig=F9fY%2FLEaVBEUvjPzODMwCMesFcBvIMnkzOjV74dhJNs%3D&jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmVsZWFzZS1hc3NldHMuZ2l0aHVidXNlcmNvbnRlbnQuY29tIiwia2V5Ijoia2V5MSIsImV4cCI6MTc2MzIyNjkzNywibmJmIjoxNzYzMjIzMzM3LCJwYXRoIjoicmVsZWFzZWFzc2V0cHJvZHVjdGlvbi5ibG9iLmNvcmUud2luZG93cy5uZXQifQ.KzIcTHG91lu0ambiXqS2gVklpB7nrFOmUT3G-VvKLls&response-content-disposition=attachment%3B%20filename%3Dgatk-4.4.0.0.zip&response-content-type=application%2Foctet-stream\n","Resolving release-assets.githubusercontent.com (release-assets.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...\n","Connecting to release-assets.githubusercontent.com (release-assets.githubusercontent.com)|185.199.108.133|:443... connected.\n","HTTP request sent, awaiting response... 200 OK\n","Length: 642466069 (613M) [application/octet-stream]\n","Saving to: ‘gatk-4.4.0.0.zip’\n","\n","gatk-4.4.0.0.zip    100%[===================>] 612.70M  94.6MB/s    in 5.9s    \n","\n","2025-11-15 16:15:43 (104 MB/s) - ‘gatk-4.4.0.0.zip’ saved [642466069/642466069]\n","\n","Archive:  gatk-4.4.0.0.zip\n","   creating: gatk-4.4.0.0/\n","   creating: gatk-4.4.0.0/gatkdoc/\n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_rnaseq_ASEReadCounter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_replicates_CollectIndependentReplicateMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_dragstr_CalibrateDragstrModel.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ChromosomeCounts.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_LibraryReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_DumpTabixIndex.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_FilterVariantTranches.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectWgsMetricsWithNonZeroCoverage.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonChimericOriginalAlignmentReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_MeanQualityByCycle.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_quality_CollectHiSeqXPfFailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_PrintReadsSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GenotypeConcordance.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CheckTerminatorBlock.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadLengthReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_Pileup.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_OverclippedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CleanSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_CreateSomaticPanelOfNormals.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_GtcToVcf.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_CheckPileup.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_BwaMemIndexImageCreator.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_HmerMotifs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_OrientationBiasReadCounts.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_RenameSampleInVcf.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_ReadAnonymizer.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappingQualityAvailableReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_CountNs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_AssemblyComplexity.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectOxoGMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_VerifyIDIntensityContaminationMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotProperlyPairedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_FingerprintingDetailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ExcessiveEndClippedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_FastaReferenceMaker.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_BarcodeMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_WellformedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_AlleleFraction.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_QualityScoreDistributionSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_DepthPerSampleHC.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_SampleReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_AnalyzeCovariates.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_RMSMappingQuality.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$FirstOfPairReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_ConvertSequencingArtifactToOxoG.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountBasesSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountReadsSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_Funcotator.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectQualityYieldMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CreateReadCountPanelOfNormals.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqPipelineSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_CalcMetadataSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$SecondOfPairReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_AlignmentAgreesWithHeaderReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_CollectDuplicateMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_StrandOddsRatio.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_filter_FilterVcf.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectWgsMetricsWithNonZeroCoverage$WgsMetricsWithNonZeroCoverage.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_ParallelCopyGCSDirectoryIntoHDFSSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_IndelErrorMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_ShiftFasta.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_varianteval_VariantEval.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_GcBiasSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectRrbsMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_RnaSeqMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_MendelianViolations_FindMendelianViolations.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotSecondaryAlignmentReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_CountReads.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_FixVcfHeader.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_CollectHsMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_AccumulateQualityYieldMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaBasecallingMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_gnarlyGenotyper_GnarlyGenotyper.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_JumpingLibraryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ValidAlignmentEndReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_EstimateLibraryComplexity.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReferenceBases.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_PrintReadsHeader.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_ViewSam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_AnnotateVcfWithExpectedAlleleFraction.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_ErrorMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_CountVariants.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_IndexFeatureFile.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$AllowAllReadsReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_LiftOverIntervalList.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_MetricsReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotDuplicateReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_FindBadGenomicKmersSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_OverlappingErrorMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fastq_BamToBfq.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_AddCommentsToBam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_QualByDepth.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_AnalyzeSaturationMutagenesis.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_CollectVariantCallingMetrics$VariantCallingSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_GenotypeGVCFs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_SequencingArtifactMetrics$PreAdapterSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_TandemRepeat.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics$ArraysControlCodesSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_BuildBamIndex.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_VariantAnnotator.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_FilterSamReads.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_BaseQualityRankSumTest.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_MergePedIntoVcf.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_Concordance.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_FragmentLengthReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_rnaseq_SplitNCigarReads.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PrimaryLineReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonZeroFragmentLengthReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_BafRegressMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectMultipleMetricsSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_reference_CompareReferences.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_StructuralVariationDiscoveryPipelineSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_EvaluateInfoFieldConcordance.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_CountBases.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_StrandOddsRatio.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_SplitVcfs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/index.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SplitSamByLibrary.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FilterFuncotations.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_MendelianViolations_MendelianViolationMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_CheckFingerprint.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNVariantTrain.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$HasReadGroupReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_CrosscheckFingerprints.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_coverage_DepthOfCoverage.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_JointGermlineCNVSegmentation.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_BaseErrorMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_PileupSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_DetermineGermlineContigPloidy.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_FindBreakpointEvidenceSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_UniqueAltReadCount.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ProperlyPairedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQuality.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$CigarContainsNoNOperator.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_plotting_PlotDenoisedCopyRatios.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_QualByDepth.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_featuremapping_FlowFeatureMapper.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_FlowBasedTPAttributeSymetricReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_VariantsToTable.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqFilterSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_VariantRecalibrator.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectBaseDistributionByCycleSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_utils_ExtractOriginalAlignmentRecordsByNameSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantrecalling_HaplotypeBasedVariantRecaller.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_reference_NormalizeFasta.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_SVConcordance.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_CycleSkipStatus.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_UnmarkDuplicates.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_HmerIndelNuc.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_RMSMappingQuality.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_quality_CollectHiSeqXPfFailMetrics$PFFailSummaryMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNVariantWriteTensors.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_SplitIntervals.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_FastaAlternateReferenceMaker.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountVariantsSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_UpdateVCFSequenceDictionary.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaBasecallsToFastq.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamFormatConverter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_PostProcessReadsForRSEM.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FisherStrand.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_rnaseq_GeneExpressionEvaluation.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_ClusterCrosscheckMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_ClipReads.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReadPosRankSumTest.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_CollectSamErrorMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_ReadGroupHasFlowOrderReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_SoftClippedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_GetSampleName.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_CollectSVEvidence.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_MappingQualityRankSumTest.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_sv_SiteDepthtoBAF.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_IntervalOverlapReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_transforms_markduplicates_MarkDuplicatesSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_DepthPerAlleleBySample.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonZeroReferenceLengthAlignmentReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CalculateReadGroupChecksum.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_groundtruth_GroundTruthReadsBuilder.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_haplotypecaller_RampedHaplotypeCaller.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_HaplotypeFilteringAnnotation.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_BaseQuality.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$SeqIsStoredReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SetNmMdAndUqTags.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_metrics_GcBiasMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CompareMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FuncotateSegments.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_GatherBamFiles.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_SequencingArtifactMetrics$BaitBiasSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_CalculateFingerprintMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FuncotatorDataSourceDownloader.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_ConvertHeaderlessHadoopBamShardToBam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBuildReferenceTaxonomy.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_FisherStrand.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_validation_CompareBaseQualities.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_replicates_IndependentReplicateMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_bwa_BwaSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadGroupBlackListReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamComparisonMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_FastqToSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectAlignmentSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_WgsMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_CollectIlluminaBasecallingMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_GcBiasDetailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SortSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_AnnotateVcfWithBamDepth.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamToFastq.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_CollectSequencingArtifactMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqScoreSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_RevertSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GenotypeConcordanceContingencyMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GatherVcfs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_SplitReads.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PassesVendorQualityCheckReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_reference_ExtractSequences.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_MarkDuplicatesWithMateCigar.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectRnaSeqMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_GatherBQSRReports.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_contamination_CalculateContamination.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_MergeSamFiles.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_ApplyBQSR.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_GatherVcfsCloud.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_inference_CpxVariantReInterpreterSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_AnnotateIntervals.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBuildKmers.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_MakeSitesOnlyVcf.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_IntervalListTools.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_InbreedingCoeff.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaPhasingMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectInsertSizeMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ReadLengthEqualsCigarLengthReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CombineGenotypingArrayVcfs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_StrandBiasBySample.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_BaitDesigner.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FragmentDepthPerAlleleBySample.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_readorientation_LearnReadOrientationModel.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_ValidateSamFile.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CollectReadCounts.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_illumina_BpmToNormalizationManifestCsv.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_FlowBasedTPAttributeValidReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_InsertSizeMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_SortSamSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_LeftAlignIndels.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_FilterIntervals.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQualityRankSumTest.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappingQualityNotZeroReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_SortVcf.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_GetNormalArtifactData.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_FixMateInformation.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_PrintVariantsSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_HaplotypeCallerSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectJumpingLibraryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SplitSamByNumberOfReads.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_HsMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_dragstr_ComposeSTRTableFile.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_MethylationTypeCaller.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectOxoGMetrics$CpcgMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateDifferentStrandReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_reference_NonNFastaSize.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_filtering_FilterMutectCalls.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_FingerprintMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_BaseRecalibratorSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_SequencingArtifactMetrics$BaitBiasDetailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_CalculateMixingFractions.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_SelectVariants.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_AccumulateVariantCallingMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaBasecallsToSam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_SVCluster.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_FlagStat.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_PrintDistantMates.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_FingerprintingSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_scalable_ScoreVariantAnnotations.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_PreprocessIntervals.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_QualityScoreDistribution.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadGroupReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_MappingQualityReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/gatkDoc.css  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_WellformedFlowBasedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectMultipleMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_CalculateGenotypePosteriors.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_genomicsdb_GenomicsDBImport.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_MergeVcfs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectQualityYieldMetricsSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_PlatformUnitReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNScoreVariants.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_RevertOriginalBaseQualitiesAndAddMateCigar.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CreateBafRegressMetricsFile.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_IndelLength.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_ExtractSVEvidenceSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_ValidateVariants.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_RawGtCount.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CollectAllelicCounts.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_CreateHadoopBamSplittingIndex.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_PrintReads.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadTagValueFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics$ArraysVariantCallingSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GenotypeConcordanceSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_LiftoverVcf.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_HmerQualitySymetricReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_BwaAndMarkDuplicatesPipelineSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_AddOATag.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_MateDistantReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_haplotypecaller_HaplotypeCaller.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_AlignmentSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_ApplyBQSRSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CreateVerifyIDIntensityContaminationMetricsFile.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReadPosition.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_Coverage.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ValidAlignmentStartReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateOnSameContigOrNoMappedMateReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_ReadsPipelineSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadStrandFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_TargetedPcrMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_ReplaceSamHeader.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_CountFalsePositives.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateUnmappedAndUnmappedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_SVAnnotate.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_TargetMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_AddOrReplaceReadGroups.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_VcfFormatConverter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_VariantType.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_BamIndexStats.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaLaneMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_Mutect2.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_CollectIlluminaLaneMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_CountBasesInReference.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_ClusteredCrosscheckMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_CombineGVCFs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_JexlExpressionReadTagValueFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectGcBiasMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_LocalAssembler.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_sv_PrintReadCounts.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_VcfToIntervalList.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SetNmAndUqTags.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBwaSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_TheoreticalSensitivityMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_plotting_PlotModeledSegments.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_MarkIlluminaAdapters.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_CollectVariantCallingMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_SampleList.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_UpdateVcfSequenceDictionary.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MatchingBasesAndQualsReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_DiscoverVariantsFromContigAlignmentsSAMSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_GermlineCNVCaller.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_MeanQualityByCycleSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FragmentLength.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_CheckIlluminaDirectory.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadNameReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_CollectTargetedPcrMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectBaseDistributionByCycle.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_ApplyVQSR.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_RrbsSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_ReblockGVCF.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_RrbsCpgDetailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_ErrorSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CallCopyRatioSegments.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_InbreedingCoeff.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_PlatformReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_ScatterIntervalsByNs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectQualityYieldMetrics$QualityYieldMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_scalable_TrainVariantAnnotationsModel.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_MergeBamAlignment.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_scalable_ExtractVariantAnnotations.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_FixMisencodedBaseQualityReads.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQualityZero.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_LeftAlignAndTrimVariants.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_ModelSegments.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FeaturizedReadSets.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_BaseRecalibrator.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_DuplicationMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_GatherTranches.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_RevertSamSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GenotypeConcordanceDetailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_ExtractIlluminaBarcodes.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_RevertBaseQualityScores.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_quality_CollectHiSeqXPfFailMetrics$PFFailDetailedMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_DownsampleSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ClippingRankSumTest.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_AmbiguousBaseReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_BedToIntervalList.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ExcessHet.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_RemoveNearbyIndels.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_filters_VariantFiltration.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_LikelihoodRankSumTest.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_validation_CompareDuplicatesSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectRawWgsMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_PossibleDeNovo.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_IndelClassify.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics$ArraysVariantCallingDetailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_TransferReadTags.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectInsertSizeMetricsSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_BaseQualityRankSumTest.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_SvDiscoverFromLocalAssemblyContigAlignmentsSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_DenoiseReadCounts.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_IntervalListToBed.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_HmerIndelLength.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CompareSAMs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_CollectVariantCallingMetrics$VariantCallingDetailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_basicshortmutpileup_ValidateBasicSomaticShortMutations.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_OriginalAlignment.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_BQSRPipelineSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_MarkDuplicates.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_UmiAwareMarkDuplicatesWithMateCigar.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_GcContent.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CreateSequenceDictionary.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_VcfToAdpc.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$GoodCigarReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_BaseDistributionByCycleMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_realignmentfilter_FilterAlignmentArtifacts.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_SimpleMarkDuplicatesWithMateCigar.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_GenotypeSummaries.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_FlagStatSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectWgsMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_UmiMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_ReorderSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectRawWgsMetrics$RawWgsMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_AllelePseudoDepth.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_PostprocessGermlineCNVCalls.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotSupplementaryAlignmentReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_ReadPosRankSumTest.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_FifoBuffer.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_PositionBasedDownsampleSam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_contamination_GetPileupSummaries.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_reference_CheckReferenceCompatibility.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PairedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_SequencingArtifactMetrics$PreAdapterDetailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_ReorderSam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectRawWgsMetrics$RawWgsMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_AllelePseudoDepth.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_PostprocessGermlineCNVCalls.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotSupplementaryAlignmentReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_BaseDistributionByCycleMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_GenotypeSummaries.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_FlagStatSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_SimpleMarkDuplicatesWithMateCigar.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_realignmentfilter_FilterAlignmentArtifacts.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectWgsMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_UmiMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_reference_CheckReferenceCompatibility.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PairedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_SequencingArtifactMetrics$PreAdapterDetailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_ReadPosRankSumTest.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_contamination_GetPileupSummaries.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_FifoBuffer.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_PositionBasedDownsampleSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_AmbiguousBaseReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_BedToIntervalList.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ExcessHet.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_RemoveNearbyIndels.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_filters_VariantFiltration.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectRawWgsMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_validation_CompareDuplicatesSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_LikelihoodRankSumTest.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_BaseRecalibrator.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_DuplicationMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_ExtractIlluminaBarcodes.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_RevertBaseQualityScores.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_RevertSamSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GenotypeConcordanceDetailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_GatherTranches.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_quality_CollectHiSeqXPfFailMetrics$PFFailDetailedMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_DownsampleSam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ClippingRankSumTest.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_basicshortmutpileup_ValidateBasicSomaticShortMutations.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_CollectVariantCallingMetrics$VariantCallingDetailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CompareSAMs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_BQSRPipelineSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_OriginalAlignment.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_UmiAwareMarkDuplicatesWithMateCigar.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_MarkDuplicates.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_VcfToAdpc.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$GoodCigarReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_GcContent.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CreateSequenceDictionary.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics$ArraysVariantCallingDetailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_TransferReadTags.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_IndelClassify.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_PossibleDeNovo.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectInsertSizeMetricsSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_DenoiseReadCounts.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_BaseQualityRankSumTest.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_SvDiscoverFromLocalAssemblyContigAlignmentsSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_HmerIndelLength.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_IntervalListToBed.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_ApplyVQSR.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_RrbsSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_ReblockGVCF.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_MeanQualityByCycleSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_DiscoverVariantsFromContigAlignmentsSAMSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_GermlineCNVCaller.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_UpdateVcfSequenceDictionary.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MatchingBasesAndQualsReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_CheckIlluminaDirectory.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FragmentLength.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_CollectTargetedPcrMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadNameReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectBaseDistributionByCycle.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_scalable_TrainVariantAnnotationsModel.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_FixMisencodedBaseQualityReads.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_scalable_ExtractVariantAnnotations.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_MergeBamAlignment.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQualityZero.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FeaturizedReadSets.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_ModelSegments.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_LeftAlignAndTrimVariants.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_RrbsCpgDetailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_ErrorSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CallCopyRatioSegments.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_ScatterIntervalsByNs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_InbreedingCoeff.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_PlatformReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectQualityYieldMetrics$QualityYieldMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaLaneMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_BamIndexStats.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_Mutect2.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_CountFalsePositives.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateUnmappedAndUnmappedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_TargetMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_SVAnnotate.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_VariantType.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_VcfFormatConverter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_AddOrReplaceReadGroups.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBwaSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SetNmAndUqTags.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_plotting_PlotModeledSegments.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_CollectVariantCallingMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_MarkIlluminaAdapters.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_TheoreticalSensitivityMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_SampleList.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_CombineGVCFs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_ClusteredCrosscheckMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_CountBasesInReference.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_CollectIlluminaLaneMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_LocalAssembler.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_sv_PrintReadCounts.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectGcBiasMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_JexlExpressionReadTagValueFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_VcfToIntervalList.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_LiftoverVcf.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GenotypeConcordanceSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics$ArraysVariantCallingSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_BwaAndMarkDuplicatesPipelineSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_HmerQualitySymetricReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_AddOATag.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_MateDistantReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_CreateHadoopBamSplittingIndex.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_PrintReads.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadTagValueFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReadPosition.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_ReadsPipelineSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ValidAlignmentStartReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateOnSameContigOrNoMappedMateReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_Coverage.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_ReplaceSamHeader.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadStrandFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_TargetedPcrMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_haplotypecaller_HaplotypeCaller.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_AlignmentSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_ApplyBQSRSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CreateVerifyIDIntensityContaminationMetricsFile.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_CalculateGenotypePosteriors.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_genomicsdb_GenomicsDBImport.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_MergeVcfs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectMultipleMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_WellformedFlowBasedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CreateBafRegressMetricsFile.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_IndelLength.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_ValidateVariants.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_RawGtCount.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_ExtractSVEvidenceSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CollectAllelicCounts.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_PlatformUnitReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectQualityYieldMetricsSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNScoreVariants.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_RevertOriginalBaseQualitiesAndAddMateCigar.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_SelectVariants.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaBasecallsToSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_AccumulateVariantCallingMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_dragstr_ComposeSTRTableFile.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectJumpingLibraryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SplitSamByNumberOfReads.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_HsMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateDifferentStrandReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_reference_NonNFastaSize.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectOxoGMetrics$CpcgMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_MethylationTypeCaller.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_BaseRecalibratorSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_filtering_FilterMutectCalls.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_FingerprintMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_CalculateMixingFractions.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_SequencingArtifactMetrics$BaitBiasDetailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_FingerprintingSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_PreprocessIntervals.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_scalable_ScoreVariantAnnotations.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_MappingQualityReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_QualityScoreDistribution.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadGroupReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_SVCluster.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_PrintDistantMates.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_FlagStat.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CombineGenotypingArrayVcfs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_BaitDesigner.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_StrandBiasBySample.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_MakeSitesOnlyVcf.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBuildKmers.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_IntervalListTools.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaPhasingMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_InbreedingCoeff.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ReadLengthEqualsCigarLengthReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectInsertSizeMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQualityRankSumTest.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_FilterIntervals.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_LeftAlignIndels.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappingQualityNotZeroReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_FixMateInformation.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_GetNormalArtifactData.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_SortVcf.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_HaplotypeCallerSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_PrintVariantsSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CollectReadCounts.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_ValidateSamFile.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_readorientation_LearnReadOrientationModel.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FragmentDepthPerAlleleBySample.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_FlowBasedTPAttributeValidReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_illumina_BpmToNormalizationManifestCsv.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_SortSamSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_InsertSizeMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_replicates_IndependentReplicateMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_FastqToSam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamComparisonMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_bwa_BwaSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadGroupBlackListReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_WgsMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectAlignmentSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamToFastq.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SortSam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_AnnotateVcfWithBamDepth.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_GcBiasDetailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_CollectIlluminaBasecallingMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_GatherBamFiles.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_SequencingArtifactMetrics$BaitBiasSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FuncotatorDataSourceDownloader.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_CalculateFingerprintMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_FisherStrand.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBuildReferenceTaxonomy.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_ConvertHeaderlessHadoopBamShardToBam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_validation_CompareBaseQualities.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_MarkDuplicatesWithMateCigar.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_MergeSamFiles.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_ApplyBQSR.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_contamination_CalculateContamination.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectRnaSeqMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_GatherBQSRReports.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_GatherVcfsCloud.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_AnnotateIntervals.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_inference_CpxVariantReInterpreterSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_CollectSequencingArtifactMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqScoreSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GenotypeConcordanceContingencyMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_RevertSam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GatherVcfs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_reference_ExtractSequences.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PassesVendorQualityCheckReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_SplitReads.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_ReadGroupHasFlowOrderReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_SoftClippedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_CollectSamErrorMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_MappingQualityRankSumTest.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_GetSampleName.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_CollectSVEvidence.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_sv_SiteDepthtoBAF.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_PostProcessReadsForRSEM.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamFormatConverter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FisherStrand.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_rnaseq_GeneExpressionEvaluation.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_ClusterCrosscheckMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_ClipReads.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReadPosRankSumTest.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_BaseQuality.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$SeqIsStoredReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SetNmMdAndUqTags.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_metrics_GcBiasMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CompareMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FuncotateSegments.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_DepthPerAlleleBySample.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_transforms_markduplicates_MarkDuplicatesSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_IntervalOverlapReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CalculateReadGroupChecksum.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonZeroReferenceLengthAlignmentReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_haplotypecaller_RampedHaplotypeCaller.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_groundtruth_GroundTruthReadsBuilder.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_HaplotypeFilteringAnnotation.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectBaseDistributionByCycleSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_VariantRecalibrator.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqFilterSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantrecalling_HaplotypeBasedVariantRecaller.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_utils_ExtractOriginalAlignmentRecordsByNameSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_SVConcordance.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_reference_NormalizeFasta.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_plotting_PlotDenoisedCopyRatios.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_QualByDepth.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_featuremapping_FlowFeatureMapper.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_FlowBasedTPAttributeSymetricReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_VariantsToTable.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_quality_CollectHiSeqXPfFailMetrics$PFFailSummaryMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_SplitIntervals.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNVariantWriteTensors.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_FastaAlternateReferenceMaker.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_UpdateVCFSequenceDictionary.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaBasecallsToFastq.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountVariantsSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_CycleSkipStatus.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_UnmarkDuplicates.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_HmerIndelNuc.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_RMSMappingQuality.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectMultipleMetricsSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_reference_CompareReferences.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_StructuralVariationDiscoveryPipelineSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_StrandOddsRatio.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_CountBases.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_EvaluateInfoFieldConcordance.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_rnaseq_SplitNCigarReads.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PrimaryLineReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonZeroFragmentLengthReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_BafRegressMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_JointGermlineCNVSegmentation.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_coverage_DepthOfCoverage.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_DetermineGermlineContigPloidy.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_FindBreakpointEvidenceSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_UniqueAltReadCount.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_PileupSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_BaseErrorMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ProperlyPairedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$CigarContainsNoNOperator.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQuality.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_SplitVcfs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FilterFuncotations.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_MendelianViolations_MendelianViolationMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SplitSamByLibrary.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_CrosscheckFingerprints.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$HasReadGroupReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_CheckFingerprint.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNVariantTrain.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_CollectVariantCallingMetrics$VariantCallingSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_GenotypeGVCFs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics$ArraysControlCodesSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_TandemRepeat.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_SequencingArtifactMetrics$PreAdapterSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_FindBadGenomicKmersSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_OverlappingErrorMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fastq_BamToBfq.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_AddCommentsToBam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_AnalyzeSaturationMutagenesis.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_QualByDepth.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_VariantAnnotator.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_FilterSamReads.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_BaseQualityRankSumTest.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_MergePedIntoVcf.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_Concordance.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_FragmentLengthReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_BuildBamIndex.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_RnaSeqMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectRrbsMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotSecondaryAlignmentReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_MendelianViolations_FindMendelianViolations.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_CollectHsMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_FixVcfHeader.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_CountReads.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectWgsMetricsWithNonZeroCoverage$WgsMetricsWithNonZeroCoverage.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_filter_FilterVcf.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_ParallelCopyGCSDirectoryIntoHDFSSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_IndelErrorMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_ShiftFasta.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_varianteval_VariantEval.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_GcBiasSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReferenceBases.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_PrintReadsHeader.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_ViewSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_AnnotateVcfWithExpectedAlleleFraction.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_IndexFeatureFile.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$AllowAllReadsReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_CountVariants.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_ErrorMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_LiftOverIntervalList.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_MetricsReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotDuplicateReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_AccumulateQualityYieldMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaBasecallingMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_JumpingLibraryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_gnarlyGenotyper_GnarlyGenotyper.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_EstimateLibraryComplexity.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ValidAlignmentEndReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_FingerprintingDetailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ExcessiveEndClippedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_BarcodeMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_FastaReferenceMaker.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_SampleReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_QualityScoreDistributionSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_DepthPerSampleHC.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_WellformedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_AlleleFraction.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_ConvertSequencingArtifactToOxoG.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$FirstOfPairReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_RMSMappingQuality.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_AnalyzeCovariates.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_OrientationBiasReadCounts.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_AssemblyComplexity.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_CountNs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_RenameSampleInVcf.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_ReadAnonymizer.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappingQualityAvailableReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectOxoGMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotProperlyPairedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_VerifyIDIntensityContaminationMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CreateReadCountPanelOfNormals.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_CalcMetadataSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqPipelineSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_CollectDuplicateMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_AlignmentAgreesWithHeaderReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$SecondOfPairReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_StrandOddsRatio.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_Funcotator.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountReadsSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountBasesSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectQualityYieldMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_MeanQualityByCycle.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_quality_CollectHiSeqXPfFailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GenotypeConcordance.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_PrintReadsSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_replicates_CollectIndependentReplicateMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_rnaseq_ASEReadCounter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ChromosomeCounts.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_dragstr_CalibrateDragstrModel.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_DumpTabixIndex.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_FilterVariantTranches.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_LibraryReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonChimericOriginalAlignmentReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectWgsMetricsWithNonZeroCoverage.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_CreateSomaticPanelOfNormals.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_CheckPileup.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_GtcToVcf.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_HmerMotifs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_BwaMemIndexImageCreator.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CheckTerminatorBlock.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadLengthReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_OverclippedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_Pileup.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CleanSam.html  \n","  inflating: gatk-4.4.0.0/gatkPythonPackageArchive.zip  \n","  inflating: gatk-4.4.0.0/gatk-completion.sh  \n","  inflating: gatk-4.4.0.0/gatk-package-4.4.0.0-spark.jar  \n","  inflating: gatk-4.4.0.0/gatkcondaenv.yml  \n","  inflating: gatk-4.4.0.0/README.md  \n","   creating: gatk-4.4.0.0/scripts/\n","   creating: gatk-4.4.0.0/scripts/sv/\n","  inflating: gatk-4.4.0.0/scripts/sv/create_cluster.sh  \n","  inflating: gatk-4.4.0.0/scripts/sv/delete_cluster.sh  \n","  inflating: gatk-4.4.0.0/scripts/sv/sanity_checks.sh  \n","  inflating: gatk-4.4.0.0/scripts/sv/manage_sv_pipeline.sh  \n","  inflating: gatk-4.4.0.0/scripts/sv/run_whole_pipeline.sh  \n","  inflating: gatk-4.4.0.0/scripts/sv/copy_sv_results.sh  \n","  inflating: gatk-4.4.0.0/scripts/sv/default_init.sh  \n","   creating: gatk-4.4.0.0/scripts/sv/stepByStep/\n","  inflating: gatk-4.4.0.0/scripts/sv/stepByStep/scanBam.sh  \n","  inflating: gatk-4.4.0.0/scripts/sv/stepByStep/discoverVariants.sh  \n","  inflating: gatk-4.4.0.0/scripts/sv/stepByStep/svDiscover.sh  \n","   creating: gatk-4.4.0.0/scripts/cnv_wdl/\n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/cnv_common_tasks.wdl  \n","   creating: gatk-4.4.0.0/scripts/cnv_wdl/germline/\n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/germline/cnv_germline_cohort_workflow.wdl  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/germline/README.md  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/germline/cnv_germline_case_workflow.wdl  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/germline/Structs.wdl  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/germline/cnv_germline_case_scattered_workflow.wdl  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/germline/joint_call_exome_cnvs.wdl  \n","   creating: gatk-4.4.0.0/scripts/cnv_wdl/somatic/\n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/somatic/cnv_somatic_oncotator_workflow.wdl  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/somatic/cnv_somatic_panel_workflow.wdl  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/somatic/README.md  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/somatic/cnv_somatic_funcotate_seg_workflow.wdl  \n","  inflating: gatk-4.4.0.0/scripts/dataproc-cluster-ui  \n","   creating: gatk-4.4.0.0/scripts/mutect2_wdl/\n","  inflating: gatk-4.4.0.0/scripts/mutect2_wdl/mutect2.wdl  \n","   creating: gatk-4.4.0.0/scripts/mutect2_wdl/mutect_resources_json/\n","  inflating: gatk-4.4.0.0/scripts/mutect2_wdl/mutect_resources_json/mutect_resources_process_gnomAD_2.1.json  \n","  inflating: gatk-4.4.0.0/scripts/mutect2_wdl/README.md  \n","  inflating: gatk-4.4.0.0/scripts/mutect2_wdl/mutect2_pon.wdl  \n","  inflating: gatk-4.4.0.0/scripts/mutect2_wdl/mutect_resources.wdl  \n","  inflating: gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar  \n","  inflating: gatk-4.4.0.0/gatk       \n","  inflating: gatk-4.4.0.0/GATKConfig.EXAMPLE.properties  \n"]}],"source":["!cd /content\n","!wget -c https://github.com/broadinstitute/gatk/releases/download/4.4.0.0/gatk-4.4.0.0.zip\n","!unzip -o gatk-4.4.0.0.zip\n"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"collapsed":true,"executionInfo":{"elapsed":157350,"status":"ok","timestamp":1763223636530,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"50MHbYgjTs5u","outputId":"54268635-dd39-412c-eeea-6bbca25fef3e"},"outputs":[{"name":"stdout","output_type":"stream","text":["--2025-11-15 16:17:58--  https://hgdownload.soe.ucsc.edu/goldenPath/hg38/chromosomes/chr12.fa.gz\n","Resolving hgdownload.soe.ucsc.edu (hgdownload.soe.ucsc.edu)... 128.114.119.163\n","Connecting to hgdownload.soe.ucsc.edu (hgdownload.soe.ucsc.edu)|128.114.119.163|:443... connected.\n","HTTP request sent, awaiting response... 200 OK\n","Length: 42940449 (41M) [application/x-gzip]\n","Saving to: ‘chr12.fa.gz’\n","\n","chr12.fa.gz         100%[===================>]  40.95M  26.8MB/s    in 1.5s    \n","\n","2025-11-15 16:18:00 (26.8 MB/s) - ‘chr12.fa.gz’ saved [42940449/42940449]\n","\n","[bwa_index] Pack FASTA... 1.07 sec\n","[bwa_index] Construct BWT for the packed sequence...\n","[BWTIncCreate] textLength=266550618, availableWord=30755168\n","[BWTIncConstructFromPacked] 10 iterations done. 50732074 characters processed.\n","[BWTIncConstructFromPacked] 20 iterations done. 93723498 characters processed.\n","[BWTIncConstructFromPacked] 30 iterations done. 131930362 characters processed.\n","[BWTIncConstructFromPacked] 40 iterations done. 165884698 characters processed.\n","[BWTIncConstructFromPacked] 50 iterations done. 196059354 characters processed.\n","[BWTIncConstructFromPacked] 60 iterations done. 222874650 characters processed.\n","[BWTIncConstructFromPacked] 70 iterations done. 246704106 characters processed.\n","[BWTIncConstructFromPacked] 80 iterations done. 266550618 characters processed.\n","[bwt_gen] Finished constructing BWT in 80 iterations.\n","[bwa_index] 98.89 seconds elapse.\n","[bwa_index] Update BWT... 0.75 sec\n","[bwa_index] Pack forward-only FASTA... 0.59 sec\n","[bwa_index] Construct SA from BWT and Occ... 42.95 sec\n","[main] Version: 0.7.17-r1188\n","[main] CMD: bwa index hg38.fa\n","[main] Real time: 147.284 sec; CPU: 144.259 sec\n","Using GATK jar /content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar\n","Running:\n","    java -Dsamjdk.use_async_io_read_samtools=false -Dsamjdk.use_async_io_write_samtools=true -Dsamjdk.use_async_io_write_tribble=false -Dsamjdk.compression_level=2 -jar /content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar CreateSequenceDictionary -R hg38.fa\n","INFO\t2025-11-15 16:20:33\tCreateSequenceDictionary\tOutput dictionary will be written in hg38.dict\n","16:20:33.975 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar!/com/intel/gkl/native/libgkl_compression.so\n","[Sat Nov 15 16:20:34 UTC 2025] CreateSequenceDictionary --REFERENCE hg38.fa --TRUNCATE_NAMES_AT_WHITESPACE true --NUM_SEQUENCES 2147483647 --VERBOSITY INFO --QUIET false --VALIDATION_STRINGENCY STRICT --COMPRESSION_LEVEL 2 --MAX_RECORDS_IN_RAM 500000 --CREATE_INDEX false --CREATE_MD5_FILE false --help false --version false --showHidden false --USE_JDK_DEFLATER false --USE_JDK_INFLATER false\n","[Sat Nov 15 16:20:34 UTC 2025] Executing as root@4451a6451ba3 on Linux 6.6.105+ amd64; OpenJDK 64-Bit Server VM 17.0.16+8-Ubuntu-0ubuntu122.04.1; Deflater: Intel; Inflater: Intel; Provider GCS is available; Picard version: Version:4.4.0.0\n","[Sat Nov 15 16:20:35 UTC 2025] picard.sam.CreateSequenceDictionary done. Elapsed time: 0.03 minutes.\n","Runtime.totalMemory()=239075328\n","Tool returned:\n","0\n"]}],"source":["!cd /content/CTC360/ref\n","\n","# Download only chromosome 12 FASTA (hg38)\n","!wget -c https://hgdownload.soe.ucsc.edu/goldenPath/hg38/chromosomes/chr12.fa.gz\n","!gunzip chr12.fa.gz\n","!mv chr12.fa hg38.fa\n","\n","# Index reference\n","!bwa index hg38.fa\n","!samtools faidx hg38.fa\n","!gatk-4.4.0.0/gatk CreateSequenceDictionary -R hg38.fa\n"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":1155738,"status":"ok","timestamp":1763224808332,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"MNUKn5GMUat_","outputId":"8ce60d18-a0ea-4c26-dffe-153934e97e82"},"outputs":[{"name":"stdout","output_type":"stream","text":["\n","2025-11-15T16:20:52 prefetch.2.11.3: Current preference is set to retrieve SRA Normalized Format files with full base quality scores.\n","2025-11-15T16:20:52 prefetch.2.11.3: 1) Downloading 'SRR13973704'...\n","2025-11-15T16:20:52 prefetch.2.11.3: SRA Normalized Format file is being retrieved, if this is different from your preference, it may be due to current file availability.\n","2025-11-15T16:20:52 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:14 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:22 prefetch.2.11.3:  'SRR13973704' is valid\n","2025-11-15T16:21:22 prefetch.2.11.3: 1) 'SRR13973704' was downloaded successfully\n","2025-11-15T16:21:27 prefetch.2.11.3: 'SRR13973704' has 25 unresolved dependencies\n","2025-11-15T16:21:27 prefetch.2.11.3: 2) Downloading 'ncbi-acc:CM000663.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:27 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:28 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:28 prefetch.2.11.3: 2) 'ncbi-acc:CM000663.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:28 prefetch.2.11.3: 3) Downloading 'ncbi-acc:CM000664.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:28 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:29 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:29 prefetch.2.11.3: 3) 'ncbi-acc:CM000664.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:29 prefetch.2.11.3: 4) Downloading 'ncbi-acc:CM000665.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:29 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:29 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:29 prefetch.2.11.3: 4) 'ncbi-acc:CM000665.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:29 prefetch.2.11.3: 5) Downloading 'ncbi-acc:CM000666.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:29 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:30 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:30 prefetch.2.11.3: 5) 'ncbi-acc:CM000666.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:30 prefetch.2.11.3: 6) Downloading 'ncbi-acc:CM000667.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:30 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:31 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:31 prefetch.2.11.3: 6) 'ncbi-acc:CM000667.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:31 prefetch.2.11.3: 7) Downloading 'ncbi-acc:CM000668.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:31 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:31 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:31 prefetch.2.11.3: 7) 'ncbi-acc:CM000668.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:31 prefetch.2.11.3: 8) Downloading 'ncbi-acc:CM000669.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:31 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:32 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:32 prefetch.2.11.3: 8) 'ncbi-acc:CM000669.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:32 prefetch.2.11.3: 9) Downloading 'ncbi-acc:CM000670.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:32 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:32 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:32 prefetch.2.11.3: 9) 'ncbi-acc:CM000670.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:32 prefetch.2.11.3: 10) Downloading 'ncbi-acc:CM000671.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:32 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:33 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:33 prefetch.2.11.3: 10) 'ncbi-acc:CM000671.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:33 prefetch.2.11.3: 11) Downloading 'ncbi-acc:CM000672.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:33 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:33 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:33 prefetch.2.11.3: 11) 'ncbi-acc:CM000672.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:33 prefetch.2.11.3: 12) Downloading 'ncbi-acc:CM000673.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:33 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:33 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:33 prefetch.2.11.3: 12) 'ncbi-acc:CM000673.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:33 prefetch.2.11.3: 13) Downloading 'ncbi-acc:CM000674.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:33 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:34 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:34 prefetch.2.11.3: 13) 'ncbi-acc:CM000674.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:34 prefetch.2.11.3: 14) Downloading 'ncbi-acc:CM000675.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:34 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:34 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:34 prefetch.2.11.3: 14) 'ncbi-acc:CM000675.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:34 prefetch.2.11.3: 15) Downloading 'ncbi-acc:CM000676.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:34 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:35 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:35 prefetch.2.11.3: 15) 'ncbi-acc:CM000676.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:35 prefetch.2.11.3: 16) Downloading 'ncbi-acc:CM000677.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:35 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:35 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:35 prefetch.2.11.3: 16) 'ncbi-acc:CM000677.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:35 prefetch.2.11.3: 17) Downloading 'ncbi-acc:CM000678.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:35 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:35 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:35 prefetch.2.11.3: 17) 'ncbi-acc:CM000678.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:35 prefetch.2.11.3: 18) Downloading 'ncbi-acc:CM000679.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:35 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:36 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:36 prefetch.2.11.3: 18) 'ncbi-acc:CM000679.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:36 prefetch.2.11.3: 19) Downloading 'ncbi-acc:CM000680.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:36 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:36 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:36 prefetch.2.11.3: 19) 'ncbi-acc:CM000680.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:36 prefetch.2.11.3: 20) Downloading 'ncbi-acc:CM000681.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:36 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:36 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:36 prefetch.2.11.3: 20) 'ncbi-acc:CM000681.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:36 prefetch.2.11.3: 21) Downloading 'ncbi-acc:CM000682.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:36 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:37 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:37 prefetch.2.11.3: 21) 'ncbi-acc:CM000682.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:37 prefetch.2.11.3: 22) Downloading 'ncbi-acc:CM000683.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:37 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:37 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:37 prefetch.2.11.3: 22) 'ncbi-acc:CM000683.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:37 prefetch.2.11.3: 23) Downloading 'ncbi-acc:CM000684.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:37 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:37 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:37 prefetch.2.11.3: 23) 'ncbi-acc:CM000684.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:37 prefetch.2.11.3: 24) Downloading 'ncbi-acc:CM000685.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:37 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:38 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:38 prefetch.2.11.3: 24) 'ncbi-acc:CM000685.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:38 prefetch.2.11.3: 25) Downloading 'ncbi-acc:CM000686.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:38 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:38 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:38 prefetch.2.11.3: 25) 'ncbi-acc:CM000686.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:38 prefetch.2.11.3: 26) Downloading 'ncbi-acc:NC_012920.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:38 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:38 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:38 prefetch.2.11.3: 26) 'ncbi-acc:NC_012920.1?vdb-ctx=refseq' was downloaded successfully\n","\n","2025-11-15T16:21:38 prefetch.2.11.3: Current preference is set to retrieve SRA Normalized Format files with full base quality scores.\n","2025-11-15T16:21:38 prefetch.2.11.3: 1) Downloading 'SRR13973878'...\n","2025-11-15T16:21:38 prefetch.2.11.3: SRA Normalized Format file is being retrieved, if this is different from your preference, it may be due to current file availability.\n","2025-11-15T16:21:38 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:42 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:43 prefetch.2.11.3:  'SRR13973878' is valid\n","2025-11-15T16:21:43 prefetch.2.11.3: 1) 'SRR13973878' was downloaded successfully\n","2025-11-15T16:21:45 prefetch.2.11.3: 'SRR13973878' has 25 unresolved dependencies\n","2025-11-15T16:21:45 prefetch.2.11.3: 2) Downloading 'ncbi-acc:CM000663.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:45 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:46 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:46 prefetch.2.11.3: 2) 'ncbi-acc:CM000663.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:46 prefetch.2.11.3: 3) Downloading 'ncbi-acc:CM000664.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:46 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:47 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:47 prefetch.2.11.3: 3) 'ncbi-acc:CM000664.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:47 prefetch.2.11.3: 4) Downloading 'ncbi-acc:CM000665.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:47 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:47 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:47 prefetch.2.11.3: 4) 'ncbi-acc:CM000665.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:47 prefetch.2.11.3: 5) Downloading 'ncbi-acc:CM000666.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:47 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:48 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:48 prefetch.2.11.3: 5) 'ncbi-acc:CM000666.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:48 prefetch.2.11.3: 6) Downloading 'ncbi-acc:CM000667.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:48 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:48 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:48 prefetch.2.11.3: 6) 'ncbi-acc:CM000667.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:48 prefetch.2.11.3: 7) Downloading 'ncbi-acc:CM000668.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:48 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:49 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:49 prefetch.2.11.3: 7) 'ncbi-acc:CM000668.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:49 prefetch.2.11.3: 8) Downloading 'ncbi-acc:CM000669.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:49 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:50 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:50 prefetch.2.11.3: 8) 'ncbi-acc:CM000669.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:50 prefetch.2.11.3: 9) Downloading 'ncbi-acc:CM000670.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:50 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:50 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:50 prefetch.2.11.3: 9) 'ncbi-acc:CM000670.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:50 prefetch.2.11.3: 10) Downloading 'ncbi-acc:CM000671.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:50 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:51 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:51 prefetch.2.11.3: 10) 'ncbi-acc:CM000671.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:51 prefetch.2.11.3: 11) Downloading 'ncbi-acc:CM000672.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:51 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:51 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:51 prefetch.2.11.3: 11) 'ncbi-acc:CM000672.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:51 prefetch.2.11.3: 12) Downloading 'ncbi-acc:CM000673.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:51 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:51 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:51 prefetch.2.11.3: 12) 'ncbi-acc:CM000673.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:51 prefetch.2.11.3: 13) Downloading 'ncbi-acc:CM000674.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:51 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:52 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:52 prefetch.2.11.3: 13) 'ncbi-acc:CM000674.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:52 prefetch.2.11.3: 14) Downloading 'ncbi-acc:CM000675.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:52 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:52 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:52 prefetch.2.11.3: 14) 'ncbi-acc:CM000675.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:52 prefetch.2.11.3: 15) Downloading 'ncbi-acc:CM000676.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:52 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:53 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:53 prefetch.2.11.3: 15) 'ncbi-acc:CM000676.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:53 prefetch.2.11.3: 16) Downloading 'ncbi-acc:CM000677.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:53 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:53 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:53 prefetch.2.11.3: 16) 'ncbi-acc:CM000677.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:53 prefetch.2.11.3: 17) Downloading 'ncbi-acc:CM000678.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:53 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:53 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:53 prefetch.2.11.3: 17) 'ncbi-acc:CM000678.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:53 prefetch.2.11.3: 18) Downloading 'ncbi-acc:CM000679.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:53 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:54 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:54 prefetch.2.11.3: 18) 'ncbi-acc:CM000679.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:54 prefetch.2.11.3: 19) Downloading 'ncbi-acc:CM000680.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:54 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:54 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:54 prefetch.2.11.3: 19) 'ncbi-acc:CM000680.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:54 prefetch.2.11.3: 20) Downloading 'ncbi-acc:CM000681.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:54 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:55 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:55 prefetch.2.11.3: 20) 'ncbi-acc:CM000681.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:55 prefetch.2.11.3: 21) Downloading 'ncbi-acc:CM000682.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:55 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:55 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:55 prefetch.2.11.3: 21) 'ncbi-acc:CM000682.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:55 prefetch.2.11.3: 22) Downloading 'ncbi-acc:CM000683.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:55 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:55 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:55 prefetch.2.11.3: 22) 'ncbi-acc:CM000683.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:55 prefetch.2.11.3: 23) Downloading 'ncbi-acc:CM000684.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:55 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:56 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:56 prefetch.2.11.3: 23) 'ncbi-acc:CM000684.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:56 prefetch.2.11.3: 24) Downloading 'ncbi-acc:CM000685.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:56 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:56 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:56 prefetch.2.11.3: 24) 'ncbi-acc:CM000685.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:56 prefetch.2.11.3: 25) Downloading 'ncbi-acc:CM000686.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:56 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:56 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:56 prefetch.2.11.3: 25) 'ncbi-acc:CM000686.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:56 prefetch.2.11.3: 26) Downloading 'ncbi-acc:NC_012920.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:56 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:57 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:57 prefetch.2.11.3: 26) 'ncbi-acc:NC_012920.1?vdb-ctx=refseq' was downloaded successfully\n","spots read      : 8,159,199\n","reads read      : 16,318,398\n","reads written   : 16,218,868\n","spots read      : 1,548,246\n","reads read      : 3,096,492\n","reads written   : 3,028,842\n"]}],"source":["!cd /content/CTC360/fastq\n","\n","# Download SRA files\n","!prefetch SRR13973704\n","!prefetch SRR13973878\n","\n","# Convert to FASTQ\n","!fasterq-dump SRR13973704 -O .\n","!fasterq-dump SRR13973878 -O .\n","\n","# Compress\n","!gzip SRR13973704_1.fastq SRR13973704_2.fastq\n","!gzip SRR13973878_1.fastq SRR13973878_2.fastq\n"]},{"cell_type":"code","execution_count":null,"metadata":{"id":"Zi3G_kHObnu2"},"outputs":[],"source":["!mv /content/SRR13973704_1.fastq.gz /content/CTC360/fastq/CTC360_pre_R1.fastq.gz\n","!mv /content/SRR13973704_2.fastq.gz /content/CTC360/fastq/CTC360_pre_R2.fastq.gz\n","\n","!mv /content/SRR13973878_1.fastq.gz /content/CTC360/fastq/CTC360_post_R1.fastq.gz\n","!mv /content/SRR13973878_2.fastq.gz /content/CTC360/fastq/CTC360_post_R2.fastq.gz\n"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"collapsed":true,"executionInfo":{"elapsed":168520,"status":"ok","timestamp":1763225775561,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"X9NC1WCibsEt","outputId":"0e9adb1c-7db5-4769-c62a-8f0815428023"},"outputs":[{"name":"stdout","output_type":"stream","text":["/content/CTC360/qc\n","Started analysis of CTC360_pre_R1.fastq.gz\n","Approx 5% complete for CTC360_pre_R1.fastq.gz\n","Approx 10% complete for CTC360_pre_R1.fastq.gz\n","Approx 15% complete for CTC360_pre_R1.fastq.gz\n","Approx 20% complete for CTC360_pre_R1.fastq.gz\n","Approx 25% complete for CTC360_pre_R1.fastq.gz\n","Approx 30% complete for CTC360_pre_R1.fastq.gz\n","Approx 35% complete for CTC360_pre_R1.fastq.gz\n","Approx 40% complete for CTC360_pre_R1.fastq.gz\n","Approx 45% complete for CTC360_pre_R1.fastq.gz\n","Approx 50% complete for CTC360_pre_R1.fastq.gz\n","Approx 55% complete for CTC360_pre_R1.fastq.gz\n","Approx 60% complete for CTC360_pre_R1.fastq.gz\n","Approx 65% complete for CTC360_pre_R1.fastq.gz\n","Approx 70% complete for CTC360_pre_R1.fastq.gz\n","Approx 75% complete for CTC360_pre_R1.fastq.gz\n","Approx 80% complete for CTC360_pre_R1.fastq.gz\n","Approx 85% complete for CTC360_pre_R1.fastq.gz\n","Approx 90% complete for CTC360_pre_R1.fastq.gz\n","Approx 95% complete for CTC360_pre_R1.fastq.gz\n","Analysis complete for CTC360_pre_R1.fastq.gz\n","Started analysis of CTC360_pre_R2.fastq.gz\n","Approx 5% complete for CTC360_pre_R2.fastq.gz\n","Approx 10% complete for CTC360_pre_R2.fastq.gz\n","Approx 15% complete for CTC360_pre_R2.fastq.gz\n","Approx 20% complete for CTC360_pre_R2.fastq.gz\n","Approx 25% complete for CTC360_pre_R2.fastq.gz\n","Approx 30% complete for CTC360_pre_R2.fastq.gz\n","Approx 35% complete for CTC360_pre_R2.fastq.gz\n","Approx 40% complete for CTC360_pre_R2.fastq.gz\n","Approx 45% complete for CTC360_pre_R2.fastq.gz\n","Approx 50% complete for CTC360_pre_R2.fastq.gz\n","Approx 55% complete for CTC360_pre_R2.fastq.gz\n","Approx 60% complete for CTC360_pre_R2.fastq.gz\n","Approx 65% complete for CTC360_pre_R2.fastq.gz\n","Approx 70% complete for CTC360_pre_R2.fastq.gz\n","Approx 75% complete for CTC360_pre_R2.fastq.gz\n","Approx 80% complete for CTC360_pre_R2.fastq.gz\n","Approx 85% complete for CTC360_pre_R2.fastq.gz\n","Approx 90% complete for CTC360_pre_R2.fastq.gz\n","Approx 95% complete for CTC360_pre_R2.fastq.gz\n","Analysis complete for CTC360_pre_R2.fastq.gz\n","Started analysis of CTC360_post_R1.fastq.gz\n","Approx 5% complete for CTC360_post_R1.fastq.gz\n","Approx 10% complete for CTC360_post_R1.fastq.gz\n","Approx 15% complete for CTC360_post_R1.fastq.gz\n","Approx 20% complete for CTC360_post_R1.fastq.gz\n","Approx 25% complete for CTC360_post_R1.fastq.gz\n","Approx 30% complete for CTC360_post_R1.fastq.gz\n","Approx 35% complete for CTC360_post_R1.fastq.gz\n","Approx 40% complete for CTC360_post_R1.fastq.gz\n","Approx 45% complete for CTC360_post_R1.fastq.gz\n","Approx 50% complete for CTC360_post_R1.fastq.gz\n","Approx 55% complete for CTC360_post_R1.fastq.gz\n","Approx 60% complete for CTC360_post_R1.fastq.gz\n","Approx 65% complete for CTC360_post_R1.fastq.gz\n","Approx 70% complete for CTC360_post_R1.fastq.gz\n","Approx 75% complete for CTC360_post_R1.fastq.gz\n","Approx 80% complete for CTC360_post_R1.fastq.gz\n","Approx 85% complete for CTC360_post_R1.fastq.gz\n","Approx 90% complete for CTC360_post_R1.fastq.gz\n","Approx 95% complete for CTC360_post_R1.fastq.gz\n","Analysis complete for CTC360_post_R1.fastq.gz\n","Started analysis of CTC360_post_R2.fastq.gz\n","Approx 5% complete for CTC360_post_R2.fastq.gz\n","Approx 10% complete for CTC360_post_R2.fastq.gz\n","Approx 15% complete for CTC360_post_R2.fastq.gz\n","Approx 20% complete for CTC360_post_R2.fastq.gz\n","Approx 25% complete for CTC360_post_R2.fastq.gz\n","Approx 30% complete for CTC360_post_R2.fastq.gz\n","Approx 35% complete for CTC360_post_R2.fastq.gz\n","Approx 40% complete for CTC360_post_R2.fastq.gz\n","Approx 45% complete for CTC360_post_R2.fastq.gz\n","Approx 50% complete for CTC360_post_R2.fastq.gz\n","Approx 55% complete for CTC360_post_R2.fastq.gz\n","Approx 60% complete for CTC360_post_R2.fastq.gz\n","Approx 65% complete for CTC360_post_R2.fastq.gz\n","Approx 70% complete for CTC360_post_R2.fastq.gz\n","Approx 75% complete for CTC360_post_R2.fastq.gz\n","Approx 80% complete for CTC360_post_R2.fastq.gz\n","Approx 85% complete for CTC360_post_R2.fastq.gz\n","Approx 90% complete for CTC360_post_R2.fastq.gz\n","Approx 95% complete for CTC360_post_R2.fastq.gz\n","Analysis complete for CTC360_post_R2.fastq.gz\n"]}],"source":["%cd /content/CTC360/qc\n","!fastqc ../fastq/CTC360_pre_R1.fastq.gz ../fastq/CTC360_pre_R2.fastq.gz \\\n","        ../fastq/CTC360_post_R1.fastq.gz ../fastq/CTC360_post_R2.fastq.gz\n"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"collapsed":true,"executionInfo":{"elapsed":284593,"status":"ok","timestamp":1763226107805,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"IDr8Ps0Jctj2","outputId":"a8a25b0d-7627-422d-8ffe-ce0bb8c3236a"},"outputs":[{"name":"stdout","output_type":"stream","text":["/content/CTC360\n","Read1 before filtering:\n","total reads: 8059669\n","total bases: 1192376576\n","Q20 bases: 1130846475(94.8397%)\n","Q30 bases: 1080570668(90.6233%)\n","\n","Read2 before filtering:\n","total reads: 8059669\n","total bases: 1191862806\n","Q20 bases: 1112656685(93.3544%)\n","Q30 bases: 1037810018(87.0746%)\n","\n","Read1 after filtering:\n","total reads: 7863888\n","total bases: 1162674436\n","Q20 bases: 1109425334(95.4201%)\n","Q30 bases: 1061472467(91.2958%)\n","\n","Read2 aftering filtering:\n","total reads: 7863888\n","total bases: 1162224972\n","Q20 bases: 1096493049(94.3443%)\n","Q30 bases: 1024828287(88.1781%)\n","\n","Filtering result:\n","reads passed filter: 15727776\n","reads failed due to low quality: 391526\n","reads failed due to too many N: 36\n","reads failed due to too short: 0\n","reads with adapter trimmed: 80234\n","bases trimmed due to adapters: 1769457\n","\n","Duplication rate: 19.3541%\n","\n","Insert size peak (evaluated by paired-end reads): 167\n","\n","JSON report: qc/pre_fastp.json\n","HTML report: qc/pre_fastp.html\n","\n","fastp -i fastq/CTC360_pre_R1.fastq.gz -I fastq/CTC360_pre_R2.fastq.gz -o trimmed/CTC360_pre_R1.trimmed.fastq.gz -O trimmed/CTC360_pre_R2.trimmed.fastq.gz --html qc/pre_fastp.html --json qc/pre_fastp.json \n","fastp v0.20.1, time used: 251 seconds\n","Read1 before filtering:\n","total reads: 1480596\n","total bases: 151574486\n","Q20 bases: 114372217(75.4561%)\n","Q30 bases: 521954(0.344355%)\n","\n","Read2 before filtering:\n","total reads: 1480596\n","total bases: 151533678\n","Q20 bases: 110299686(72.7889%)\n","Q30 bases: 410892(0.271156%)\n","\n","Read1 after filtering:\n","total reads: 1265594\n","total bases: 126296283\n","Q20 bases: 104999366(83.1373%)\n","Q30 bases: 48378(0.0383052%)\n","\n","Read2 aftering filtering:\n","total reads: 1265594\n","total bases: 126209736\n","Q20 bases: 101321880(80.2806%)\n","Q30 bases: 46986(0.0372285%)\n","\n","Filtering result:\n","reads passed filter: 2531188\n","reads failed due to low quality: 428094\n","reads failed due to too many N: 1910\n","reads failed due to too short: 0\n","reads with adapter trimmed: 101156\n","bases trimmed due to adapters: 1222739\n","\n","Duplication rate: 0.895107%\n","\n","Insert size peak (evaluated by paired-end reads): 86\n","\n","JSON report: qc/post_fastp.json\n","HTML report: qc/post_fastp.html\n","\n","fastp -i fastq/CTC360_post_R1.fastq.gz -I fastq/CTC360_post_R2.fastq.gz -o trimmed/CTC360_post_R1.trimmed.fastq.gz -O trimmed/CTC360_post_R2.trimmed.fastq.gz --html qc/post_fastp.html --json qc/post_fastp.json \n","fastp v0.20.1, time used: 33 seconds\n"]}],"source":["%cd /content/CTC360\n","\n","!fastp \\\n","  -i fastq/CTC360_pre_R1.fastq.gz \\\n","  -I fastq/CTC360_pre_R2.fastq.gz \\\n","  -o trimmed/CTC360_pre_R1.trimmed.fastq.gz \\\n","  -O trimmed/CTC360_pre_R2.trimmed.fastq.gz \\\n","  --html qc/pre_fastp.html \\\n","  --json qc/pre_fastp.json\n","\n","!fastp \\\n","  -i fastq/CTC360_post_R1.fastq.gz \\\n","  -I fastq/CTC360_post_R2.fastq.gz \\\n","  -o trimmed/CTC360_post_R1.trimmed.fastq.gz \\\n","  -O trimmed/CTC360_post_R2.trimmed.fastq.gz \\\n","  --html qc/post_fastp.html \\\n","  --json qc/post_fastp.json\n"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"collapsed":true,"executionInfo":{"elapsed":14311309,"status":"ok","timestamp":1763240539985,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"DbVIoNOmeD4p","outputId":"900c4c9d-2b25-4132-da7c-ed1bb14298ce"},"outputs":[{"name":"stdout","output_type":"stream","text":["[M::bwa_idx_load_from_disk] read 0 ALT contigs\n","[M::process] read 541552 sequences (80000010 bp)...\n","[M::process] read 541052 sequences (80000234 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 1525, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (29, 68, 102)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 248)\n","[M::mem_pestat] mean and std.dev: (72.48, 47.52)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 321)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 541552 reads in 1053.462 CPU sec, 628.793 real sec\n","[M::process] read 540758 sequences (80000200 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (3, 8158, 1, 1)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (76, 106, 132)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 244)\n","[M::mem_pestat] mean and std.dev: (105.34, 38.00)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 300)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 541052 reads in 961.081 CPU sec, 572.491 real sec\n","[M::process] read 539034 sequences (80000113 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 1272, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (45, 102, 150)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 360)\n","[M::mem_pestat] mean and std.dev: (102.45, 60.87)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 465)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 540758 reads in 828.075 CPU sec, 493.628 real sec\n","[M::process] read 542666 sequences (80000162 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 257, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (24, 49, 95)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 237)\n","[M::mem_pestat] mean and std.dev: (62.61, 41.66)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 308)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 539034 reads in 170.151 CPU sec, 102.275 real sec\n","[M::process] read 541866 sequences (80000002 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 2317, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (64, 64, 64)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (64, 64)\n","[M::mem_pestat] mean and std.dev: (64.00, 0.00)\n","[M::mem_pestat] low and high boundaries for proper pairs: (64, 64)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 542666 reads in 350.375 CPU sec, 214.412 real sec\n","[M::process] read 540282 sequences (80000067 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 2454, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (31, 72, 129)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 325)\n","[M::mem_pestat] mean and std.dev: (80.46, 53.72)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 423)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 541866 reads in 671.877 CPU sec, 398.395 real sec\n","[M::process] read 541452 sequences (80000022 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 3566, 1, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (87, 147, 231)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 519)\n","[M::mem_pestat] mean and std.dev: (156.45, 108.59)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 663)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 540282 reads in 690.687 CPU sec, 408.460 real sec\n","[M::process] read 539246 sequences (80000194 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 6533, 0, 1)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (65, 143, 170)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 380)\n","[M::mem_pestat] mean and std.dev: (127.92, 72.32)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 485)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 541452 reads in 1388.833 CPU sec, 824.430 real sec\n","[M::process] read 541472 sequences (80000006 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 2929, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (71, 71, 71)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (71, 71)\n","[M::mem_pestat] mean and std.dev: (71.00, 0.00)\n","[M::mem_pestat] low and high boundaries for proper pairs: (71, 71)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 539246 reads in 229.742 CPU sec, 145.044 real sec\n","[M::process] read 541218 sequences (80000054 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 402, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (33, 58, 133)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 333)\n","[M::mem_pestat] mean and std.dev: (77.94, 56.51)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 433)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 541472 reads in 341.584 CPU sec, 201.938 real sec\n","[M::process] read 541258 sequences (80000002 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 1, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] skip orientation FR as there are not enough pairs\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 541218 reads in 90.934 CPU sec, 55.349 real sec\n","[M::process] read 540550 sequences (80000161 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 4857, 1, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (99, 122, 151)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 255)\n","[M::mem_pestat] mean and std.dev: (114.79, 40.21)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 307)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 541258 reads in 951.182 CPU sec, 564.398 real sec\n","[M::process] read 540520 sequences (80000218 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 5344, 1, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (103, 159, 186)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 352)\n","[M::mem_pestat] mean and std.dev: (142.87, 67.56)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 435)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 540550 reads in 750.310 CPU sec, 444.108 real sec\n","[M::process] read 540568 sequences (80000070 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 2278, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (139, 158, 171)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (75, 235)\n","[M::mem_pestat] mean and std.dev: (157.62, 24.75)\n","[M::mem_pestat] low and high boundaries for proper pairs: (43, 267)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 540520 reads in 399.752 CPU sec, 245.698 real sec\n","[M::process] read 539966 sequences (80000152 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 303, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (44, 87, 159)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 389)\n","[M::mem_pestat] mean and std.dev: (103.28, 70.80)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 504)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 540568 reads in 288.300 CPU sec, 170.173 real sec\n","[M::process] read 540758 sequences (80000279 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 8523, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (23, 23, 23)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (23, 23)\n","[M::mem_pestat] mean and std.dev: (23.00, 0.00)\n","[M::mem_pestat] low and high boundaries for proper pairs: (23, 23)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 539966 reads in 104.096 CPU sec, 61.965 real sec\n","[M::process] read 540798 sequences (80000117 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 420, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (76, 84, 104)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (20, 160)\n","[M::mem_pestat] mean and std.dev: (82.55, 31.22)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 207)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 540758 reads in 429.877 CPU sec, 254.232 real sec\n","[M::process] read 541490 sequences (80000275 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 2331, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (24, 28, 142)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 378)\n","[M::mem_pestat] mean and std.dev: (77.96, 69.64)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 496)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 540798 reads in 446.641 CPU sec, 271.617 real sec\n","[M::process] read 541956 sequences (80000182 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 2309, 2, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (81, 140, 164)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 330)\n","[M::mem_pestat] mean and std.dev: (124.02, 53.08)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 413)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 541490 reads in 834.678 CPU sec, 496.507 real sec\n","[M::process] read 541120 sequences (80000217 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 4362, 1, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (66, 105, 159)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 345)\n","[M::mem_pestat] mean and std.dev: (92.26, 49.55)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 438)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 541956 reads in 1249.471 CPU sec, 740.597 real sec\n","[M::process] read 541960 sequences (80000231 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 2119, 1, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (24, 24, 27)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (18, 33)\n","[M::mem_pestat] mean and std.dev: (24.07, 0.62)\n","[M::mem_pestat] low and high boundaries for proper pairs: (15, 36)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 541120 reads in 484.225 CPU sec, 287.565 real sec\n","[M::process] read 540256 sequences (80000229 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 110768, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (158, 171, 200)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (74, 284)\n","[M::mem_pestat] mean and std.dev: (167.95, 26.99)\n","[M::mem_pestat] low and high boundaries for proper pairs: (32, 326)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 541960 reads in 836.641 CPU sec, 503.248 real sec\n","[M::process] read 542608 sequences (80000184 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 263045, 2, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (159, 171, 196)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (85, 270)\n","[M::mem_pestat] mean and std.dev: (168.40, 23.71)\n","[M::mem_pestat] low and high boundaries for proper pairs: (48, 307)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 540256 reads in 97.770 CPU sec, 58.374 real sec\n","[M::process] read 541630 sequences (80000183 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 94883, 2, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (158, 172, 214)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (46, 326)\n","[M::mem_pestat] mean and std.dev: (181.38, 50.60)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 384)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 542608 reads in 1200.704 CPU sec, 709.237 real sec\n","[M::process] read 542168 sequences (80000158 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 3534, 1, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (23, 25, 25)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (19, 29)\n","[M::mem_pestat] mean and std.dev: (24.16, 1.00)\n","[M::mem_pestat] low and high boundaries for proper pairs: (17, 31)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 541630 reads in 556.242 CPU sec, 329.450 real sec\n","[M::process] read 541412 sequences (80000226 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 1839, 2, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (61, 120, 160)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 358)\n","[M::mem_pestat] mean and std.dev: (114.88, 62.66)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 457)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 542168 reads in 1485.422 CPU sec, 887.504 real sec\n","[M::process] read 542792 sequences (80000021 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 117, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (110, 135, 146)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (38, 218)\n","[M::mem_pestat] mean and std.dev: (130.67, 31.14)\n","[M::mem_pestat] low and high boundaries for proper pairs: (2, 255)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 541412 reads in 246.467 CPU sec, 147.560 real sec\n","[M::process] read 542202 sequences (80000158 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 3159, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (24, 68, 142)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 378)\n","[M::mem_pestat] mean and std.dev: (83.13, 62.26)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 496)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 542792 reads in 1690.151 CPU sec, 1000.869 real sec\n","[M::process] read 33166 sequences (4895281 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 7100, 1, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (79, 125, 181)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 385)\n","[M::mem_pestat] mean and std.dev: (132.09, 71.86)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 487)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 542202 reads in 1558.477 CPU sec, 923.426 real sec\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 213, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (73, 138, 162)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 340)\n","[M::mem_pestat] mean and std.dev: (126.51, 67.97)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 429)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 33166 reads in 105.244 CPU sec, 71.444 real sec\n","[main] Version: 0.7.17-r1188\n","[main] CMD: bwa mem -t 8 /content/hg38.fa /content/CTC360/trimmed/CTC360_pre_R1.trimmed.fastq.gz /content/CTC360/trimmed/CTC360_pre_R2.trimmed.fastq.gz\n","[main] Real time: 12218.175 sec; CPU: 20495.541 sec\n","[bam_sort_core] merging from 7 files and 1 in-memory blocks...\n","[M::bwa_idx_load_from_disk] read 0 ALT contigs\n","[M::process] read 804994 sequences (80000048 bp)...\n","[M::process] read 795874 sequences (80000018 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 15481, 1, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (31, 42, 57)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 109)\n","[M::mem_pestat] mean and std.dev: (45.15, 18.44)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 135)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 804994 reads in 1008.958 CPU sec, 600.030 real sec\n","[M::process] read 804064 sequences (80000004 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 14240, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (29, 40, 56)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 110)\n","[M::mem_pestat] mean and std.dev: (44.37, 19.01)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 137)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 795874 reads in 962.545 CPU sec, 573.032 real sec\n","[M::process] read 126256 sequences (12505949 bp)...\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 64939, 1, 3)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (53, 71, 92)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 170)\n","[M::mem_pestat] mean and std.dev: (73.65, 29.23)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 209)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 804064 reads in 1126.866 CPU sec, 666.219 real sec\n","[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 2377, 0, 0)\n","[M::mem_pestat] skip orientation FF as there are not enough pairs\n","[M::mem_pestat] analyzing insert size distribution for orientation FR...\n","[M::mem_pestat] (25, 50, 75) percentile: (31, 43, 60)\n","[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 118)\n","[M::mem_pestat] mean and std.dev: (47.35, 20.77)\n","[M::mem_pestat] low and high boundaries for proper pairs: (1, 147)\n","[M::mem_pestat] skip orientation RF as there are not enough pairs\n","[M::mem_pestat] skip orientation RR as there are not enough pairs\n","[M::mem_process_seqs] Processed 126256 reads in 184.368 CPU sec, 111.131 real sec\n","[main] Version: 0.7.17-r1188\n","[main] CMD: bwa mem -t 8 /content/hg38.fa /content/CTC360/trimmed/CTC360_post_R1.trimmed.fastq.gz /content/CTC360/trimmed/CTC360_post_R2.trimmed.fastq.gz\n","[main] Real time: 1954.157 sec; CPU: 3285.356 sec\n"]}],"source":["!bwa mem -t 8 /content/hg38.fa \\\n","  /content/CTC360/trimmed/CTC360_pre_R1.trimmed.fastq.gz \\\n","  /content/CTC360/trimmed/CTC360_pre_R2.trimmed.fastq.gz \\\n","  | samtools sort -o /content/CTC360/bam/CTC360_pre.bam\n","\n","!bwa mem -t 8 /content/hg38.fa \\\n","  /content/CTC360/trimmed/CTC360_post_R1.trimmed.fastq.gz \\\n","  /content/CTC360/trimmed/CTC360_post_R2.trimmed.fastq.gz \\\n","  | samtools sort -o /content/CTC360/bam/CTC360_post.bam\n"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":114,"status":"ok","timestamp":1763240565588,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"pdymOJsSeOzm","outputId":"e6d1f5ad-4d28-4e83-a7c3-870086db36d4"},"outputs":[{"name":"stdout","output_type":"stream","text":["total 1.6G\n","-rw-r--r-- 1 root root 249M Nov 15 21:02 CTC360_post.bam\n","-rw-r--r-- 1 root root 1.4G Nov 15 20:29 CTC360_pre.bam\n"]}],"source":["!ls -lh bam"]},{"cell_type":"code","execution_count":null,"metadata":{"id":"8utnzo7KU9qC"},"outputs":[],"source":["!samtools index /content/CTC360/bam/CTC360_pre.bam\n","!samtools index /content/CTC360/bam/CTC360_post.bam"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"collapsed":true,"executionInfo":{"elapsed":237547,"status":"ok","timestamp":1763240872707,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"thp6zkuYVMIM","outputId":"cbf57bad-4c77-4dde-af7f-29d7206bbffb"},"outputs":[{"name":"stdout","output_type":"stream","text":["/content/CTC360\n","Using GATK jar /content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar\n","Running:\n","    java -Dsamjdk.use_async_io_read_samtools=false -Dsamjdk.use_async_io_write_samtools=true -Dsamjdk.use_async_io_write_tribble=false -Dsamjdk.compression_level=2 -jar /content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar MarkDuplicates -I bam/CTC360_pre.bam -O bam/CTC360_pre.dedup.bam -M bam/CTC360_pre.metrics --REMOVE_DUPLICATES true\n","21:04:00.215 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar!/com/intel/gkl/native/libgkl_compression.so\n","[Sat Nov 15 21:04:00 UTC 2025] MarkDuplicates --INPUT bam/CTC360_pre.bam --OUTPUT bam/CTC360_pre.dedup.bam --METRICS_FILE bam/CTC360_pre.metrics --REMOVE_DUPLICATES true --MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP 50000 --MAX_FILE_HANDLES_FOR_READ_ENDS_MAP 8000 --SORTING_COLLECTION_SIZE_RATIO 0.25 --TAG_DUPLICATE_SET_MEMBERS false --REMOVE_SEQUENCING_DUPLICATES false --TAGGING_POLICY DontTag --CLEAR_DT true --DUPLEX_UMI false --FLOW_MODE false --FLOW_QUALITY_SUM_STRATEGY false --USE_END_IN_UNPAIRED_READS false --USE_UNPAIRED_CLIPPED_END false --UNPAIRED_END_UNCERTAINTY 0 --FLOW_SKIP_FIRST_N_FLOWS 0 --FLOW_Q_IS_KNOWN_END false --FLOW_EFFECTIVE_QUALITY_THRESHOLD 15 --ADD_PG_TAG_TO_READS true --ASSUME_SORTED false --DUPLICATE_SCORING_STRATEGY SUM_OF_BASE_QUALITIES --PROGRAM_RECORD_ID MarkDuplicates --PROGRAM_GROUP_NAME MarkDuplicates --READ_NAME_REGEX <optimized capture of last three ':' separated fields as numeric values> --OPTICAL_DUPLICATE_PIXEL_DISTANCE 100 --MAX_OPTICAL_DUPLICATE_SET_SIZE 300000 --VERBOSITY INFO --QUIET false --VALIDATION_STRINGENCY STRICT --COMPRESSION_LEVEL 2 --MAX_RECORDS_IN_RAM 500000 --CREATE_INDEX false --CREATE_MD5_FILE false --help false --version false --showHidden false --USE_JDK_DEFLATER false --USE_JDK_INFLATER false\n","[Sat Nov 15 21:04:00 UTC 2025] Executing as root@4451a6451ba3 on Linux 6.6.105+ amd64; OpenJDK 64-Bit Server VM 17.0.16+8-Ubuntu-0ubuntu122.04.1; Deflater: Intel; Inflater: Intel; Provider GCS is available; Picard version: Version:4.4.0.0\n","INFO\t2025-11-15 21:04:00\tMarkDuplicates\tStart of doWork freeMemory: 83019504; totalMemory: 113246208; maxMemory: 3401580544\n","INFO\t2025-11-15 21:04:00\tMarkDuplicates\tReading input file and constructing read end information.\n","INFO\t2025-11-15 21:04:00\tMarkDuplicates\tWill retain up to 12324567 data points before spilling to disk.\n","WARNING\t2025-11-15 21:04:00\tAbstractOpticalDuplicateFinderCommandLineProgram\tA field field parsed out of a read name was expected to contain an integer and did not. Read name: SRR13973704.5845352. Cause: String 'SRR13973704.5845352' did not start with a parsable number.\n","INFO\t2025-11-15 21:04:12\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:   11s.  Last read position: chr12:15,681,448\n","INFO\t2025-11-15 21:04:12\tMarkDuplicates\tTracking 125835 as yet unmatched pairs. 125835 records in RAM.\n","INFO\t2025-11-15 21:04:21\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:20s.  Time for last 1,000,000:    8s.  Last read position: chr12:36,472,975\n","INFO\t2025-11-15 21:04:21\tMarkDuplicates\tTracking 201709 as yet unmatched pairs. 201709 records in RAM.\n","INFO\t2025-11-15 21:04:28\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:27s.  Time for last 1,000,000:    7s.  Last read position: chr12:55,068,125\n","INFO\t2025-11-15 21:04:28\tMarkDuplicates\tTracking 261837 as yet unmatched pairs. 261837 records in RAM.\n","INFO\t2025-11-15 21:04:35\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:35s.  Time for last 1,000,000:    7s.  Last read position: chr12:62,179,671\n","INFO\t2025-11-15 21:04:35\tMarkDuplicates\tTracking 276633 as yet unmatched pairs. 276633 records in RAM.\n","INFO\t2025-11-15 21:04:42\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:41s.  Time for last 1,000,000:    6s.  Last read position: chr12:86,309,119\n","INFO\t2025-11-15 21:04:42\tMarkDuplicates\tTracking 272248 as yet unmatched pairs. 272248 records in RAM.\n","INFO\t2025-11-15 21:04:51\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:51s.  Time for last 1,000,000:    9s.  Last read position: chr12:106,820,933\n","INFO\t2025-11-15 21:04:51\tMarkDuplicates\tTracking 228157 as yet unmatched pairs. 228157 records in RAM.\n","INFO\t2025-11-15 21:04:58\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:57s.  Time for last 1,000,000:    6s.  Last read position: chr12:119,715,155\n","INFO\t2025-11-15 21:04:58\tMarkDuplicates\tTracking 142158 as yet unmatched pairs. 142158 records in RAM.\n","INFO\t2025-11-15 21:05:05\tMarkDuplicates\tRead 7987168 records. 0 pairs never matched.\n","INFO\t2025-11-15 21:05:06\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 1141414336; totalMemory: 2025848832; maxMemory: 3401580544\n","INFO\t2025-11-15 21:05:06\tMarkDuplicates\tWill retain up to 106299392 duplicate indices before spilling to disk.\n","INFO\t2025-11-15 21:05:08\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n","INFO\t2025-11-15 21:05:13\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n","INFO\t2025-11-15 21:05:15\tMarkDuplicates\tSorting list of duplicate records.\n","INFO\t2025-11-15 21:05:16\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 2074998664; totalMemory: 2956984320; maxMemory: 3401580544\n","INFO\t2025-11-15 21:05:16\tMarkDuplicates\tMarking 2193335 records as duplicates.\n","INFO\t2025-11-15 21:05:16\tMarkDuplicates\tFound 0 optical duplicate clusters.\n","INFO\t2025-11-15 21:05:16\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n","INFO\t2025-11-15 21:06:28\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:01:11s.  Time for last 10,000,000:   71s.  Last read position: */*\n","INFO\t2025-11-15 21:06:50\tMarkDuplicates\tWriting complete. Closing input iterator.\n","INFO\t2025-11-15 21:06:50\tMarkDuplicates\tDuplicate Index cleanup.\n","INFO\t2025-11-15 21:06:50\tMarkDuplicates\tGetting Memory Stats.\n","INFO\t2025-11-15 21:06:50\tMarkDuplicates\tBefore output close freeMemory: 87131648; totalMemory: 119537664; maxMemory: 3401580544\n","INFO\t2025-11-15 21:06:50\tMarkDuplicates\tClosed outputs. Getting more Memory Stats.\n","INFO\t2025-11-15 21:06:50\tMarkDuplicates\tAfter output close freeMemory: 88691376; totalMemory: 119537664; maxMemory: 3401580544\n","[Sat Nov 15 21:06:50 UTC 2025] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 2.84 minutes.\n","Runtime.totalMemory()=119537664\n","Tool returned:\n","0\n","Using GATK jar /content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar\n","Running:\n","    java -Dsamjdk.use_async_io_read_samtools=false -Dsamjdk.use_async_io_write_samtools=true -Dsamjdk.use_async_io_write_tribble=false -Dsamjdk.compression_level=2 -jar /content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar MarkDuplicates -I bam/CTC360_post.bam -O bam/CTC360_post.dedup.bam -M bam/CTC360_post.metrics --REMOVE_DUPLICATES true\n","21:06:53.914 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar!/com/intel/gkl/native/libgkl_compression.so\n","[Sat Nov 15 21:06:53 UTC 2025] MarkDuplicates --INPUT bam/CTC360_post.bam --OUTPUT bam/CTC360_post.dedup.bam --METRICS_FILE bam/CTC360_post.metrics --REMOVE_DUPLICATES true --MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP 50000 --MAX_FILE_HANDLES_FOR_READ_ENDS_MAP 8000 --SORTING_COLLECTION_SIZE_RATIO 0.25 --TAG_DUPLICATE_SET_MEMBERS false --REMOVE_SEQUENCING_DUPLICATES false --TAGGING_POLICY DontTag --CLEAR_DT true --DUPLEX_UMI false --FLOW_MODE false --FLOW_QUALITY_SUM_STRATEGY false --USE_END_IN_UNPAIRED_READS false --USE_UNPAIRED_CLIPPED_END false --UNPAIRED_END_UNCERTAINTY 0 --FLOW_SKIP_FIRST_N_FLOWS 0 --FLOW_Q_IS_KNOWN_END false --FLOW_EFFECTIVE_QUALITY_THRESHOLD 15 --ADD_PG_TAG_TO_READS true --ASSUME_SORTED false --DUPLICATE_SCORING_STRATEGY SUM_OF_BASE_QUALITIES --PROGRAM_RECORD_ID MarkDuplicates --PROGRAM_GROUP_NAME MarkDuplicates --READ_NAME_REGEX <optimized capture of last three ':' separated fields as numeric values> --OPTICAL_DUPLICATE_PIXEL_DISTANCE 100 --MAX_OPTICAL_DUPLICATE_SET_SIZE 300000 --VERBOSITY INFO --QUIET false --VALIDATION_STRINGENCY STRICT --COMPRESSION_LEVEL 2 --MAX_RECORDS_IN_RAM 500000 --CREATE_INDEX false --CREATE_MD5_FILE false --help false --version false --showHidden false --USE_JDK_DEFLATER false --USE_JDK_INFLATER false\n","[Sat Nov 15 21:06:53 UTC 2025] Executing as root@4451a6451ba3 on Linux 6.6.105+ amd64; OpenJDK 64-Bit Server VM 17.0.16+8-Ubuntu-0ubuntu122.04.1; Deflater: Intel; Inflater: Intel; Provider GCS is available; Picard version: Version:4.4.0.0\n","INFO\t2025-11-15 21:06:54\tMarkDuplicates\tStart of doWork freeMemory: 83006744; totalMemory: 113246208; maxMemory: 3401580544\n","INFO\t2025-11-15 21:06:54\tMarkDuplicates\tReading input file and constructing read end information.\n","INFO\t2025-11-15 21:06:54\tMarkDuplicates\tWill retain up to 12324567 data points before spilling to disk.\n","WARNING\t2025-11-15 21:06:54\tAbstractOpticalDuplicateFinderCommandLineProgram\tA field field parsed out of a read name was expected to contain an integer and did not. Read name: SRR13973878.1. Cause: String 'SRR13973878.1' did not start with a parsable number.\n","INFO\t2025-11-15 21:07:04\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:10s.  Time for last 1,000,000:   10s.  Last read position: chr12:65,410,800\n","INFO\t2025-11-15 21:07:04\tMarkDuplicates\tTracking 14576 as yet unmatched pairs. 14576 records in RAM.\n","INFO\t2025-11-15 21:07:10\tMarkDuplicates\tRead 1962309 records. 0 pairs never matched.\n","INFO\t2025-11-15 21:07:11\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 707530872; totalMemory: 1038090240; maxMemory: 3401580544\n","INFO\t2025-11-15 21:07:11\tMarkDuplicates\tWill retain up to 106299392 duplicate indices before spilling to disk.\n","INFO\t2025-11-15 21:07:11\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n","INFO\t2025-11-15 21:07:13\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n","INFO\t2025-11-15 21:07:13\tMarkDuplicates\tSorting list of duplicate records.\n","INFO\t2025-11-15 21:07:13\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 1005458568; totalMemory: 1887436800; maxMemory: 3401580544\n","INFO\t2025-11-15 21:07:13\tMarkDuplicates\tMarking 19551 records as duplicates.\n","INFO\t2025-11-15 21:07:13\tMarkDuplicates\tFound 0 optical duplicate clusters.\n","INFO\t2025-11-15 21:07:13\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n","INFO\t2025-11-15 21:07:32\tMarkDuplicates\tWriting complete. Closing input iterator.\n","INFO\t2025-11-15 21:07:32\tMarkDuplicates\tDuplicate Index cleanup.\n","INFO\t2025-11-15 21:07:32\tMarkDuplicates\tGetting Memory Stats.\n","INFO\t2025-11-15 21:07:32\tMarkDuplicates\tBefore output close freeMemory: 81207400; totalMemory: 113246208; maxMemory: 3401580544\n","INFO\t2025-11-15 21:07:32\tMarkDuplicates\tClosed outputs. Getting more Memory Stats.\n","INFO\t2025-11-15 21:07:32\tMarkDuplicates\tAfter output close freeMemory: 82410200; totalMemory: 113246208; maxMemory: 3401580544\n","[Sat Nov 15 21:07:32 UTC 2025] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 0.65 minutes.\n","Runtime.totalMemory()=113246208\n","Tool returned:\n","0\n","total 3.7G\n","-rw-r--r-- 1 root root 249M Nov 15 21:02 CTC360_post.bam\n","-rw-r--r-- 1 root root 154K Nov 15 21:03 CTC360_post.bam.bai\n","-rw-r--r-- 1 root root 329M Nov 15 21:07 CTC360_post.dedup.bam\n","-rw-r--r-- 1 root root 161K Nov 15 21:07 CTC360_post.dedup.bam.bai\n","-rw-r--r-- 1 root root 3.6K Nov 15 21:07 CTC360_post.metrics\n","-rw-r--r-- 1 root root 1.4G Nov 15 20:29 CTC360_pre.bam\n","-rw-r--r-- 1 root root 228K Nov 15 21:03 CTC360_pre.bam.bai\n","-rw-r--r-- 1 root root 1.8G Nov 15 21:06 CTC360_pre.dedup.bam\n","-rw-r--r-- 1 root root 246K Nov 15 21:07 CTC360_pre.dedup.bam.bai\n","-rw-r--r-- 1 root root 4.0K Nov 15 21:06 CTC360_pre.metrics\n"]}],"source":["%cd /content/CTC360\n","\n","!/content/gatk-4.4.0.0/gatk MarkDuplicates \\\n","  -I bam/CTC360_pre.bam \\\n","  -O bam/CTC360_pre.dedup.bam \\\n","  -M bam/CTC360_pre.metrics \\\n","  --REMOVE_DUPLICATES true\n","\n","!/content/gatk-4.4.0.0/gatk MarkDuplicates \\\n","  -I bam/CTC360_post.bam \\\n","  -O bam/CTC360_post.dedup.bam \\\n","  -M bam/CTC360_post.metrics \\\n","  --REMOVE_DUPLICATES true\n","\n","!samtools index bam/CTC360_pre.dedup.bam\n","!samtools index bam/CTC360_post.dedup.bam\n","\n","!ls -lh bam\n"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"collapsed":true,"executionInfo":{"elapsed":117,"status":"ok","timestamp":1763241411681,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"QNSAMx47WMYt","outputId":"a06c83d1-4aec-41a0-84ea-949b737bf333"},"outputs":[{"name":"stdout","output_type":"stream","text":["-rw-r--r-- 1 root root  462 Nov 15 16:19 /content/hg38.fa.amb\n","-rw-r--r-- 1 root root   45 Nov 15 16:19 /content/hg38.fa.ann\n","-rw-r--r-- 1 root root 128M Nov 15 16:19 /content/hg38.fa.bwt\n","-rw-r--r-- 1 root root  32M Nov 15 16:19 /content/hg38.fa.pac\n","-rw-r--r-- 1 root root  64M Nov 15 16:20 /content/hg38.fa.sa\n"]}],"source":["!ls -lh /content/hg38.fa*\n"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":113,"status":"ok","timestamp":1763241593479,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"EfaHEa9qYVmW","outputId":"100910e9-6b85-44f6-9a4b-03e6846e931b"},"outputs":[{"name":"stdout","output_type":"stream","text":["total 130M\n","drwxr-xr-x 2 root root 4.0K Nov 15 21:07 bam\n","drwxr-xr-x 2 root root 4.0K Nov 15 16:56 fastq\n","drwxr-xr-x 2 root root 4.0K Nov 15 16:13 figures\n","-rw-r--r-- 1 root root   98 Nov 15 16:20 hg38.dict\n","-rw-r--r-- 1 root root 130M Jan 24  2014 hg38.fa\n","-rw-r--r-- 1 root root   24 Nov 15 16:20 hg38.fa.fai\n","drwxr-xr-x 2 root root 4.0K Nov 15 16:47 qc\n","drwxr-xr-x 2 root root 4.0K Nov 15 16:13 ref\n","drwxr-xr-x 2 root root 4.0K Nov 15 17:01 trimmed\n","drwxr-xr-x 2 root root 4.0K Nov 15 16:13 variants\n"]}],"source":["\n","!ls -lh /content/CTC360\n"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":115,"status":"ok","timestamp":1763241810256,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"MOZ-gTdPZs8Q","outputId":"c0246de0-137c-4f85-df0f-acfc5c8420f6"},"outputs":[{"name":"stdout","output_type":"stream","text":["total 0\n"]}],"source":["!ls -lh /content/CTC360/variants\n"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":115,"status":"ok","timestamp":1763241884684,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"uzbgouL1Z4Ji","outputId":"08677721-d427-495e-cef0-2b5f0c5a3608"},"outputs":[{"name":"stdout","output_type":"stream","text":["-rw-r--r-- 1 root root 130M Jan 24  2014 /content/CTC360/hg38.fa\n"]}],"source":["!ls -lh /content/CTC360/hg38.fa\n"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":134,"status":"ok","timestamp":1763242033255,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"fAXPf4owaLH7","outputId":"a115fbcb-1ebd-4392-aae0-aa916f471dba"},"outputs":[{"name":"stdout","output_type":"stream","text":["total 0\n"]}],"source":["!ls -lh /content/CTC360/variants/\n"]},{"cell_type":"code","execution_count":null,"metadata":{"id":"hIEkDiXVatIz"},"outputs":[],"source":[]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":108,"status":"ok","timestamp":1763242130868,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"Z2ScjJ9hatoD","outputId":"69b6e3eb-759b-4f4a-d3aa-d02b97fe2fda"},"outputs":[{"name":"stdout","output_type":"stream","text":["@HD\tVN:1.6\tSO:coordinate\n","@SQ\tSN:chr12\tLN:133275309\n","@PG\tID:bwa\tPN:bwa\tVN:0.7.17-r1188\tCL:bwa mem -t 8 /content/hg38.fa /content/CTC360/trimmed/CTC360_pre_R1.trimmed.fastq.gz /content/CTC360/trimmed/CTC360_pre_R2.trimmed.fastq.gz\n","@PG\tID:samtools\tPN:samtools\tPP:bwa\tVN:1.13\tCL:samtools sort -o /content/CTC360/bam/CTC360_pre.bam\n","@PG\tID:MarkDuplicates\tVN:Version:4.4.0.0\tCL:MarkDuplicates --INPUT bam/CTC360_pre.bam --OUTPUT bam/CTC360_pre.dedup.bam --METRICS_FILE bam/CTC360_pre.metrics --REMOVE_DUPLICATES true --MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP 50000 --MAX_FILE_HANDLES_FOR_READ_ENDS_MAP 8000 --SORTING_COLLECTION_SIZE_RATIO 0.25 --TAG_DUPLICATE_SET_MEMBERS false --REMOVE_SEQUENCING_DUPLICATES false --TAGGING_POLICY DontTag --CLEAR_DT true --DUPLEX_UMI false --FLOW_MODE false --FLOW_QUALITY_SUM_STRATEGY false --USE_END_IN_UNPAIRED_READS false --USE_UNPAIRED_CLIPPED_END false --UNPAIRED_END_UNCERTAINTY 0 --FLOW_SKIP_FIRST_N_FLOWS 0 --FLOW_Q_IS_KNOWN_END false --FLOW_EFFECTIVE_QUALITY_THRESHOLD 15 --ADD_PG_TAG_TO_READS true --ASSUME_SORTED false --DUPLICATE_SCORING_STRATEGY SUM_OF_BASE_QUALITIES --PROGRAM_RECORD_ID MarkDuplicates --PROGRAM_GROUP_NAME MarkDuplicates --READ_NAME_REGEX <optimized capture of last three ':' separated fields as numeric values> --OPTICAL_DUPLICATE_PIXEL_DISTANCE 100 --MAX_OPTICAL_DUPLICATE_SET_SIZE 300000 --VERBOSITY INFO --QUIET false --VALIDATION_STRINGENCY STRICT --COMPRESSION_LEVEL 2 --MAX_RECORDS_IN_RAM 500000 --CREATE_INDEX false --CREATE_MD5_FILE false --help false --version false --showHidden false --USE_JDK_DEFLATER false --USE_JDK_INFLATER false\tPN:MarkDuplicates\n","@PG\tID:samtools.1\tPN:samtools\tPP:samtools\tVN:1.13\tCL:samtools view -H /content/CTC360/bam/CTC360_pre.dedup.bam\n"]}],"source":["!samtools view -H /content/CTC360/bam/CTC360_pre.dedup.bam\n"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"collapsed":true,"executionInfo":{"elapsed":106806,"status":"ok","timestamp":1763242298108,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"3eNH-jW9bEQK","outputId":"e5ee1952-0991-4157-b276-9c2edabc0f0a"},"outputs":[{"name":"stdout","output_type":"stream","text":["Using GATK jar /content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar\n","Running:\n","    java -Dsamjdk.use_async_io_read_samtools=false -Dsamjdk.use_async_io_write_samtools=true -Dsamjdk.use_async_io_write_tribble=false -Dsamjdk.compression_level=2 -jar /content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar AddOrReplaceReadGroups -I /content/CTC360/bam/CTC360_pre.dedup.bam -O /content/CTC360/bam/CTC360_pre.RG.bam -RGID PRE -RGLB lib1 -RGPL ILLUMINA -RGPU unit1 -RGSM PRE\n","21:29:53.897 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar!/com/intel/gkl/native/libgkl_compression.so\n","[Sat Nov 15 21:29:53 UTC 2025] AddOrReplaceReadGroups --INPUT /content/CTC360/bam/CTC360_pre.dedup.bam --OUTPUT /content/CTC360/bam/CTC360_pre.RG.bam --RGID PRE --RGLB lib1 --RGPL ILLUMINA --RGPU unit1 --RGSM PRE --VERBOSITY INFO --QUIET false --VALIDATION_STRINGENCY STRICT --COMPRESSION_LEVEL 2 --MAX_RECORDS_IN_RAM 500000 --CREATE_INDEX false --CREATE_MD5_FILE false --help false --version false --showHidden false --USE_JDK_DEFLATER false --USE_JDK_INFLATER false\n","[Sat Nov 15 21:29:53 UTC 2025] Executing as root@4451a6451ba3 on Linux 6.6.105+ amd64; OpenJDK 64-Bit Server VM 17.0.16+8-Ubuntu-0ubuntu122.04.1; Deflater: Intel; Inflater: Intel; Provider GCS is available; Picard version: Version:4.4.0.0\n","INFO\t2025-11-15 21:29:54\tAddOrReplaceReadGroups\tCreated read-group ID=PRE PL=ILLUMINA LB=lib1 SM=PRE\n","\n","INFO\t2025-11-15 21:30:05\tAddOrReplaceReadGroups\tProcessed     1,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:   11s.  Last read position: chr12:25,189,707\n","INFO\t2025-11-15 21:30:14\tAddOrReplaceReadGroups\tProcessed     2,000,000 records.  Elapsed time: 00:00:20s.  Time for last 1,000,000:    8s.  Last read position: chr12:50,798,112\n","INFO\t2025-11-15 21:30:23\tAddOrReplaceReadGroups\tProcessed     3,000,000 records.  Elapsed time: 00:00:29s.  Time for last 1,000,000:    8s.  Last read position: chr12:67,081,167\n","INFO\t2025-11-15 21:30:32\tAddOrReplaceReadGroups\tProcessed     4,000,000 records.  Elapsed time: 00:00:38s.  Time for last 1,000,000:    9s.  Last read position: chr12:99,466,390\n","INFO\t2025-11-15 21:30:40\tAddOrReplaceReadGroups\tProcessed     5,000,000 records.  Elapsed time: 00:00:46s.  Time for last 1,000,000:    7s.  Last read position: chr12:118,573,933\n","INFO\t2025-11-15 21:30:50\tAddOrReplaceReadGroups\tProcessed     6,000,000 records.  Elapsed time: 00:00:56s.  Time for last 1,000,000:    9s.  Last read position: */*\n","INFO\t2025-11-15 21:30:56\tAddOrReplaceReadGroups\tProcessed     7,000,000 records.  Elapsed time: 00:01:02s.  Time for last 1,000,000:    5s.  Last read position: */*\n","INFO\t2025-11-15 21:31:02\tAddOrReplaceReadGroups\tProcessed     8,000,000 records.  Elapsed time: 00:01:08s.  Time for last 1,000,000:    6s.  Last read position: */*\n","INFO\t2025-11-15 21:31:07\tAddOrReplaceReadGroups\tProcessed     9,000,000 records.  Elapsed time: 00:01:13s.  Time for last 1,000,000:    5s.  Last read position: */*\n","INFO\t2025-11-15 21:31:13\tAddOrReplaceReadGroups\tProcessed    10,000,000 records.  Elapsed time: 00:01:19s.  Time for last 1,000,000:    5s.  Last read position: */*\n","INFO\t2025-11-15 21:31:19\tAddOrReplaceReadGroups\tProcessed    11,000,000 records.  Elapsed time: 00:01:25s.  Time for last 1,000,000:    5s.  Last read position: */*\n","INFO\t2025-11-15 21:31:24\tAddOrReplaceReadGroups\tProcessed    12,000,000 records.  Elapsed time: 00:01:30s.  Time for last 1,000,000:    4s.  Last read position: */*\n","INFO\t2025-11-15 21:31:30\tAddOrReplaceReadGroups\tProcessed    13,000,000 records.  Elapsed time: 00:01:36s.  Time for last 1,000,000:    6s.  Last read position: */*\n","INFO\t2025-11-15 21:31:35\tAddOrReplaceReadGroups\tProcessed    14,000,000 records.  Elapsed time: 00:01:41s.  Time for last 1,000,000:    4s.  Last read position: */*\n","[Sat Nov 15 21:31:37 UTC 2025] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 1.72 minutes.\n","Runtime.totalMemory()=322961408\n","Tool returned:\n","0\n"]}],"source":["!/content/gatk-4.4.0.0/gatk AddOrReplaceReadGroups \\\n","    -I /content/CTC360/bam/CTC360_pre.dedup.bam \\\n","    -O /content/CTC360/bam/CTC360_pre.RG.bam \\\n","    -RGID PRE \\\n","    -RGLB lib1 \\\n","    -RGPL ILLUMINA \\\n","    -RGPU unit1 \\\n","    -RGSM PRE\n"]},{"cell_type":"code","execution_count":null,"metadata":{"id":"lZpb9tmkb2DX"},"outputs":[],"source":["!samtools index /content/CTC360/bam/CTC360_pre.RG.bam\n","\n"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":123,"status":"ok","timestamp":1763242422107,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"},"user_tz":300},"id":"BDPl_5vucCRZ","outputId":"833572c6-dc1c-47dc-a8c3-b8ffdd34324c"},"outputs":[{"name":"stdout","output_type":"stream","text":["@RG\tID:PRE\tLB:lib1\tPL:ILLUMINA\tSM:PRE\tPU:unit1\n"]}],"source":["!samtools view -H /content/CTC360/bam/CTC360_pre.RG.bam | grep \"@RG\"\n"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"background_save":true,"base_uri":"https://localhost:8080/"},"collapsed":true,"id":"G8lejgs9cMzI","outputId":"a6e67c4b-35b0-4f87-c772-91c73e71606a"},"outputs":[{"name":"stdout","output_type":"stream","text":["Using GATK jar /content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar\n","Running:\n","    java -Dsamjdk.use_async_io_read_samtools=false -Dsamjdk.use_async_io_write_samtools=true -Dsamjdk.use_async_io_write_tribble=false -Dsamjdk.compression_level=2 -jar /content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar Mutect2 -R /content/CTC360/hg38.fa -I /content/CTC360/bam/CTC360_pre.RG.bam -tumor PRE -O /content/CTC360/variants/CTC360_pre.unfiltered.vcf.gz\n","22:06:33.006 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar!/com/intel/gkl/native/libgkl_compression.so\n","22:06:33.092 INFO  Mutect2 - ------------------------------------------------------------\n","22:06:33.099 INFO  Mutect2 - The Genome Analysis Toolkit (GATK) v4.4.0.0\n","22:06:33.099 INFO  Mutect2 - For support and documentation go to https://software.broadinstitute.org/gatk/\n","22:06:33.099 INFO  Mutect2 - Executing as root@4451a6451ba3 on Linux v6.6.105+ amd64\n","22:06:33.099 INFO  Mutect2 - Java runtime: OpenJDK 64-Bit Server VM v17.0.16+8-Ubuntu-0ubuntu122.04.1\n","22:06:33.100 INFO  Mutect2 - Start Date/Time: November 15, 2025 at 10:06:32 PM UTC\n","22:06:33.100 INFO  Mutect2 - ------------------------------------------------------------\n","22:06:33.100 INFO  Mutect2 - ------------------------------------------------------------\n","22:06:33.102 INFO  Mutect2 - HTSJDK Version: 3.0.5\n","22:06:33.104 INFO  Mutect2 - Picard Version: 3.0.0\n","22:06:33.104 INFO  Mutect2 - Built for Spark Version: 3.3.1\n","22:06:33.105 INFO  Mutect2 - HTSJDK Defaults.COMPRESSION_LEVEL : 2\n","22:06:33.105 INFO  Mutect2 - HTSJDK Defaults.USE_ASYNC_IO_READ_FOR_SAMTOOLS : false\n","22:06:33.105 INFO  Mutect2 - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_SAMTOOLS : true\n","22:06:33.107 INFO  Mutect2 - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_TRIBBLE : false\n","22:06:33.108 INFO  Mutect2 - Deflater: IntelDeflater\n","22:06:33.108 INFO  Mutect2 - Inflater: IntelInflater\n","22:06:33.108 INFO  Mutect2 - GCS max retries/reopens: 20\n","22:06:33.108 INFO  Mutect2 - Requester pays: disabled\n","22:06:33.109 INFO  Mutect2 - Initializing engine\n","22:06:33.334 INFO  Mutect2 - Done initializing engine\n","22:06:33.379 INFO  NativeLibraryLoader - Loading libgkl_utils.so from jar:file:/content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar!/com/intel/gkl/native/libgkl_utils.so\n","22:06:33.393 INFO  NativeLibraryLoader - Loading libgkl_pairhmm_omp.so from jar:file:/content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar!/com/intel/gkl/native/libgkl_pairhmm_omp.so\n","22:06:33.410 INFO  IntelPairHmm - Flush-to-zero (FTZ) is enabled when running PairHMM\n","22:06:33.411 INFO  IntelPairHmm - Available threads: 2\n","22:06:33.411 INFO  IntelPairHmm - Requested threads: 4\n","22:06:33.411 WARN  IntelPairHmm - Using 2 available threads, but 4 were requested\n","22:06:33.412 INFO  PairHMM - Using the OpenMP multi-threaded AVX-accelerated native PairHMM implementation\n","22:06:33.475 INFO  ProgressMeter - Starting traversal\n","22:06:33.476 INFO  ProgressMeter -        Current Locus  Elapsed Minutes     Regions Processed   Regions/Minute\n","22:06:43.534 INFO  ProgressMeter -          chr12:54151              0.2                   250           1491.8\n","22:06:54.149 INFO  ProgressMeter -         chr12:217612              0.3                   960           2786.5\n","22:07:04.280 INFO  ProgressMeter -         chr12:311724              0.5                  1350           2629.5\n","22:07:14.282 INFO  ProgressMeter -         chr12:368780              0.7                  1610           2367.3\n","22:07:24.295 INFO  ProgressMeter -         chr12:417877              0.8                  1850           2184.2\n","22:07:34.706 INFO  ProgressMeter -         chr12:514418              1.0                  2300           2253.8\n","22:07:44.800 INFO  ProgressMeter -         chr12:614442              1.2                  2750           2313.4\n","22:07:55.088 INFO  ProgressMeter -         chr12:656414              1.4                  2940           2161.4\n","22:08:05.518 INFO  ProgressMeter -         chr12:708829              1.5                  3200           2086.0\n","22:08:16.715 INFO  ProgressMeter -         chr12:744858              1.7                  3370           1958.6\n","22:08:27.619 INFO  ProgressMeter -         chr12:851449              1.9                  3850           2023.8\n","22:08:39.596 INFO  ProgressMeter -         chr12:942986              2.1                  4270           2031.4\n","22:08:50.254 INFO  ProgressMeter -         chr12:985893              2.3                  4490           1969.6\n","22:09:00.705 INFO  ProgressMeter -        chr12:1031510              2.5                  4710           1919.5\n","22:09:11.408 INFO  ProgressMeter -        chr12:1128560              2.6                  5140           1952.7\n","22:09:21.466 INFO  ProgressMeter -        chr12:1219658              2.8                  5530           1975.1\n","22:09:31.840 INFO  ProgressMeter -        chr12:1343123              3.0                  6050           2035.2\n","22:09:41.912 INFO  ProgressMeter -        chr12:1549949              3.1                  6940           2209.8\n","22:09:52.613 INFO  ProgressMeter -        chr12:1642192              3.3                  7340           2211.5\n","22:10:02.900 INFO  ProgressMeter -        chr12:1688160              3.5                  7550           2163.1\n","22:10:13.205 INFO  ProgressMeter -        chr12:1817584              3.7                  8120           2217.3\n","22:10:23.791 INFO  ProgressMeter -        chr12:1942875              3.8                  8650           2253.4\n","22:10:34.070 INFO  ProgressMeter -        chr12:2118722              4.0                  9370           2336.7\n","22:10:44.267 INFO  ProgressMeter -        chr12:2578636              4.2                 11200           2679.5\n","22:10:56.981 INFO  ProgressMeter -        chr12:2717901              4.4                 11790           2684.6\n","22:11:07.030 INFO  ProgressMeter -        chr12:2793209              4.6                 12110           2656.2\n","22:11:17.469 INFO  ProgressMeter -        chr12:2853548              4.7                 12380           2615.6\n","22:11:27.668 INFO  ProgressMeter -        chr12:2921090              4.9                 12720           2594.2\n","22:11:37.791 INFO  ProgressMeter -        chr12:2957618              5.1                 12910           2545.4\n","22:11:48.010 INFO  ProgressMeter -        chr12:3014108              5.2                 13180           2514.2\n","22:11:58.252 INFO  ProgressMeter -        chr12:3065501              5.4                 13430           2481.1\n","22:12:33.662 INFO  ProgressMeter -        chr12:3097204              6.0                 13590           2263.8\n","22:12:43.691 INFO  ProgressMeter -        chr12:3290780              6.2                 14410           2335.4\n","22:12:53.854 INFO  ProgressMeter -        chr12:3458714              6.3                 15100           2381.8\n","22:13:03.973 INFO  ProgressMeter -        chr12:3769726              6.5                 16390           2518.3\n","22:13:15.838 INFO  ProgressMeter -        chr12:3911144              6.7                 16980           2532.0\n","22:13:26.887 INFO  ProgressMeter -        chr12:4009664              6.9                 17410           2526.8\n","22:13:36.903 INFO  ProgressMeter -        chr12:4195555              7.1                 18230           2583.2\n","22:13:47.862 INFO  ProgressMeter -        chr12:4369748              7.2                 18980           2621.6\n","22:13:59.716 INFO  ProgressMeter -        chr12:4456326              7.4                 19360           2603.1\n","22:14:09.916 INFO  ProgressMeter -        chr12:4668854              7.6                 20230           2659.3\n","22:14:20.457 INFO  ProgressMeter -        chr12:4725506              7.8                 20470           2630.1\n","22:14:30.906 INFO  ProgressMeter -        chr12:5117706              8.0                 22070           2773.6\n","22:14:40.914 INFO  ProgressMeter -        chr12:5458775              8.1                 23430           2884.1\n","22:14:51.439 INFO  ProgressMeter -        chr12:5790468              8.3                 24770           2984.6\n","22:15:11.921 INFO  ProgressMeter -        chr12:5823479              8.6                 24900           2881.7\n","22:15:22.086 INFO  ProgressMeter -        chr12:6014829              8.8                 25710           2918.2\n","22:15:34.053 INFO  ProgressMeter -        chr12:6069157              9.0                 25970           2882.5\n","22:15:44.630 INFO  ProgressMeter -        chr12:6133198              9.2                 26260           2858.7\n","22:15:55.835 INFO  ProgressMeter -        chr12:6196183              9.4                 26550           2832.7\n","22:16:06.186 INFO  ProgressMeter -        chr12:6263174              9.5                 26860           2814.0\n","22:16:16.467 INFO  ProgressMeter -        chr12:6319130              9.7                 27130           2792.2\n","22:16:26.538 INFO  ProgressMeter -        chr12:6437315              9.9                 27700           2802.4\n","22:16:36.642 INFO  ProgressMeter -        chr12:6475417             10.1                 27890           2774.4\n","22:16:46.839 INFO  ProgressMeter -        chr12:6485883             10.2                 27940           2733.1\n","22:16:59.728 INFO  ProgressMeter -        chr12:6538339             10.4                 28180           2699.9\n","22:17:10.204 INFO  ProgressMeter -        chr12:6599857             10.6                 28490           2684.7\n","22:17:20.320 INFO  ProgressMeter -        chr12:6658211             10.8                 28770           2668.6\n","22:17:30.505 INFO  ProgressMeter -        chr12:6742446             11.0                 29160           2662.9\n","22:17:41.597 INFO  ProgressMeter -        chr12:6807507             11.1                 29470           2646.5\n","22:17:53.238 INFO  ProgressMeter -        chr12:6884390             11.3                 29820           2632.1\n","22:18:03.452 INFO  ProgressMeter -        chr12:6969670             11.5                 30220           2627.9\n","22:18:13.466 INFO  ProgressMeter -        chr12:7026143             11.7                 30470           2611.8\n","22:18:24.189 INFO  ProgressMeter -        chr12:7096991             11.8                 30780           2598.5\n","22:18:41.852 INFO  ProgressMeter -        chr12:7200814             12.1                 31240           2573.4\n","22:18:53.157 INFO  ProgressMeter -        chr12:7325922             12.3                 31760           2576.2\n","22:19:03.160 INFO  ProgressMeter -        chr12:7517386             12.5                 32540           2604.3\n","22:19:15.194 INFO  ProgressMeter -        chr12:7590308             12.7                 32840           2586.8\n","22:19:28.205 INFO  ProgressMeter -        chr12:7592792             12.9                 32850           2544.1\n","22:19:38.331 INFO  ProgressMeter -        chr12:7655267             13.1                 33120           2531.9\n","22:19:48.525 INFO  ProgressMeter -        chr12:7684662             13.3                 33280           2511.5\n","22:19:59.458 INFO  ProgressMeter -        chr12:7737271             13.4                 33560           2498.3\n","22:20:09.476 INFO  ProgressMeter -        chr12:7771601             13.6                 33740           2480.9\n","22:20:19.826 INFO  ProgressMeter -        chr12:7831180             13.8                 34040           2471.6\n","22:20:30.867 INFO  ProgressMeter -        chr12:7868208             14.0                 34220           2451.9\n","22:20:40.908 INFO  ProgressMeter -        chr12:7902622             14.1                 34390           2434.9\n","22:20:51.332 INFO  ProgressMeter -        chr12:7945449             14.3                 34600           2420.0\n","22:21:01.509 INFO  ProgressMeter -        chr12:7987987             14.5                 34810           2406.1\n","22:21:14.289 INFO  ProgressMeter -        chr12:8025310             14.7                 35000           2384.2\n","22:21:25.659 INFO  ProgressMeter -        chr12:8107472             14.9                 35370           2378.7\n","22:21:39.850 INFO  ProgressMeter -        chr12:8184423             15.1                 35700           2363.3\n","22:21:50.639 INFO  ProgressMeter -        chr12:8246290             15.3                 35980           2353.8\n","22:22:01.271 INFO  ProgressMeter -        chr12:8263912             15.5                 36080           2333.3\n","22:22:11.406 INFO  ProgressMeter -        chr12:8280449             15.6                 36180           2314.5\n","22:22:21.870 INFO  ProgressMeter -        chr12:8314086             15.8                 36340           2299.0\n","22:22:33.389 INFO  ProgressMeter -        chr12:8335032             16.0                 36450           2278.3\n","22:22:43.453 INFO  ProgressMeter -        chr12:8358587             16.2                 36570           2262.1\n","22:22:55.694 INFO  ProgressMeter -        chr12:8373069             16.4                 36650           2238.8\n","22:23:05.869 INFO  ProgressMeter -        chr12:8389930             16.5                 36730           2220.7\n","22:23:16.250 INFO  ProgressMeter -        chr12:8433680             16.7                 36930           2209.7\n","22:23:29.326 INFO  ProgressMeter -        chr12:8463439             16.9                 37060           2188.9\n","22:23:39.334 INFO  ProgressMeter -        chr12:8616825             17.1                 37740           2207.3\n","22:24:59.123 INFO  ProgressMeter -        chr12:8636226             18.4                 37840           2053.5\n","22:25:28.076 INFO  ProgressMeter -        chr12:8638139             18.9                 37850           2001.6\n","22:25:39.150 INFO  ProgressMeter -        chr12:8723041             19.1                 38230           2002.1\n","22:25:49.682 INFO  ProgressMeter -        chr12:8822242             19.3                 38670           2006.7\n","22:26:01.393 INFO  ProgressMeter -        chr12:8885096             19.5                 38960           2001.5\n","22:26:12.112 INFO  ProgressMeter -        chr12:8971536             19.6                 39330           2002.1\n","22:26:23.593 INFO  ProgressMeter -        chr12:9171690             19.8                 40190           2026.2\n","22:26:33.714 INFO  ProgressMeter -        chr12:9485625             20.0                 41410           2070.1\n","22:27:36.546 INFO  ProgressMeter -        chr12:9696830             21.1                 42220           2005.6\n","22:27:46.786 INFO  ProgressMeter -        chr12:9918348             21.2                 43120           2031.9\n","22:27:56.932 INFO  ProgressMeter -       chr12:10137635             21.4                 44010           2057.4\n","22:28:07.512 INFO  ProgressMeter -       chr12:10190294             21.6                 44260           2052.2\n","22:28:19.592 INFO  ProgressMeter -       chr12:10245211             21.8                 44510           2044.7\n","22:28:29.942 INFO  ProgressMeter -       chr12:10343002             21.9                 44920           2047.3\n","22:28:45.742 INFO  ProgressMeter -       chr12:10573783             22.2                 45830           2064.0\n","22:28:56.382 INFO  ProgressMeter -       chr12:10898507             22.4                 47150           2106.6\n","22:29:21.203 INFO  ProgressMeter -       chr12:11459264             22.8                 49340           2164.5\n","22:29:32.652 INFO  ProgressMeter -       chr12:11606628             23.0                 49970           2173.9\n","22:29:43.137 INFO  ProgressMeter -       chr12:11867710             23.2                 51030           2203.3\n","22:29:53.173 INFO  ProgressMeter -       chr12:12023579             23.3                 51730           2217.5\n","22:30:05.529 INFO  ProgressMeter -       chr12:12091066             23.5                 52050           2211.7\n","22:30:15.960 INFO  ProgressMeter -       chr12:12236597             23.7                 52710           2223.3\n","22:30:27.159 INFO  ProgressMeter -       chr12:12289425             23.9                 52970           2216.8\n","22:30:37.421 INFO  ProgressMeter -       chr12:12313418             24.1                 53090           2206.0\n","22:30:48.698 INFO  ProgressMeter -       chr12:12413975             24.3                 53570           2208.7\n","22:30:59.974 INFO  ProgressMeter -       chr12:12420426             24.4                 53600           2193.0\n","22:31:10.950 INFO  ProgressMeter -       chr12:12548315             24.6                 54140           2198.6\n","22:31:24.590 INFO  ProgressMeter -       chr12:12566445             24.9                 54220           2181.7\n","22:31:38.671 INFO  ProgressMeter -       chr12:12619404             25.1                 54470           2171.3\n","22:31:48.787 INFO  ProgressMeter -       chr12:12713933             25.3                 54890           2173.4\n","22:31:58.807 INFO  ProgressMeter -       chr12:12798561             25.4                 55280           2174.5\n","22:32:10.528 INFO  ProgressMeter -       chr12:12857136             25.6                 55570           2169.2\n","22:32:21.318 INFO  ProgressMeter -       chr12:12922444             25.8                 55880           2166.1\n","22:32:31.383 INFO  ProgressMeter -       chr12:13038122             26.0                 56410           2172.5\n","22:32:41.385 INFO  ProgressMeter -       chr12:13329212             26.1                 57590           2203.8\n","22:32:52.498 INFO  ProgressMeter -       chr12:13645691             26.3                 58870           2237.0\n","22:33:02.555 INFO  ProgressMeter -       chr12:13993624             26.5                 60240           2274.5\n","22:33:12.836 INFO  ProgressMeter -       chr12:14166294             26.7                 60970           2287.3\n","22:33:22.839 INFO  ProgressMeter -       chr12:14247657             26.8                 61320           2286.1\n","22:33:37.391 INFO  ProgressMeter -       chr12:14295342             27.1                 61550           2274.1\n","22:34:12.004 INFO  ProgressMeter -       chr12:14355305             27.6                 61840           2237.2\n","22:34:22.130 INFO  ProgressMeter -       chr12:14574260             27.8                 62780           2257.4\n","22:34:32.278 INFO  ProgressMeter -       chr12:14736385             28.0                 63490           2269.1\n","22:34:42.555 INFO  ProgressMeter -       chr12:14939007             28.2                 64300           2284.1\n","22:34:52.591 INFO  ProgressMeter -       chr12:15316296             28.3                 65840           2325.0\n","22:35:02.603 INFO  ProgressMeter -       chr12:15570895             28.5                 66870           2347.5\n","22:35:12.764 INFO  ProgressMeter -       chr12:15835108             28.7                 67960           2371.7\n","22:35:23.402 INFO  ProgressMeter -       chr12:15945871             28.8                 68420           2373.0\n","22:35:33.460 INFO  ProgressMeter -       chr12:16183072             29.0                 69380           2392.4\n","22:35:44.787 INFO  ProgressMeter -       chr12:16343289             29.2                 70030           2399.2\n","22:35:55.361 INFO  ProgressMeter -       chr12:16697980             29.4                 71420           2432.2\n","22:36:05.439 INFO  ProgressMeter -       chr12:17101955             29.5                 72990           2471.5\n","22:36:15.447 INFO  ProgressMeter -       chr12:17480527             29.7                 74490           2508.1\n","22:36:25.494 INFO  ProgressMeter -       chr12:17760098             29.9                 75590           2530.9\n","22:36:35.964 INFO  ProgressMeter -       chr12:18127956             30.0                 77030           2564.1\n","22:36:48.073 INFO  ProgressMeter -       chr12:18318782             30.2                 77790           2572.1\n","22:36:58.081 INFO  ProgressMeter -       chr12:18656869             30.4                 79120           2601.8\n","22:37:08.277 INFO  ProgressMeter -       chr12:19051677             30.6                 80700           2639.0\n","22:37:18.421 INFO  ProgressMeter -       chr12:19109184             30.7                 80950           2632.6\n","22:37:28.764 INFO  ProgressMeter -       chr12:19326977             30.9                 81870           2647.7\n","22:37:38.781 INFO  ProgressMeter -       chr12:19380952             31.1                 82160           2642.8\n","22:37:49.329 INFO  ProgressMeter -       chr12:19485618             31.3                 82650           2643.6\n","22:37:59.335 INFO  ProgressMeter -       chr12:19672629             31.4                 83490           2656.3\n","22:38:09.810 INFO  ProgressMeter -       chr12:19908537             31.6                 84460           2672.3\n","22:38:19.895 INFO  ProgressMeter -       chr12:20048743             31.8                 85030           2676.1\n","22:38:29.995 INFO  ProgressMeter -       chr12:20452172             31.9                 86640           2712.4\n","22:38:40.187 INFO  ProgressMeter -       chr12:20768072             32.1                 87880           2736.7\n","22:38:50.199 INFO  ProgressMeter -       chr12:21223840             32.3                 89710           2779.2\n","22:39:00.339 INFO  ProgressMeter -       chr12:21544147             32.4                 90990           2804.2\n","22:39:10.491 INFO  ProgressMeter -       chr12:21704218             32.6                 91640           2809.6\n","22:39:20.845 INFO  ProgressMeter -       chr12:21990705             32.8                 92780           2829.6\n","22:39:31.347 INFO  ProgressMeter -       chr12:22135861             33.0                 93400           2833.3\n","22:39:41.389 INFO  ProgressMeter -       chr12:22380414             33.1                 94430           2850.1\n","22:39:52.520 INFO  ProgressMeter -       chr12:22587274             33.3                 95260           2859.2\n","22:40:02.794 INFO  ProgressMeter -       chr12:22773614             33.5                 96010           2866.9\n","22:40:12.895 INFO  ProgressMeter -       chr12:23035776             33.7                 97070           2884.1\n","22:40:22.912 INFO  ProgressMeter -       chr12:23419414             33.8                 98590           2914.8\n","22:40:34.127 INFO  ProgressMeter -       chr12:23737574             34.0                 99840           2935.5\n","22:40:44.139 INFO  ProgressMeter -       chr12:24144611             34.2                101410           2967.1\n","22:40:54.887 INFO  ProgressMeter -       chr12:24438989             34.4                102580           2985.7\n","22:41:12.465 INFO  ProgressMeter -       chr12:24673936             34.6                103540           2988.2\n","22:41:22.661 INFO  ProgressMeter -       chr12:24900735             34.8                104480           3000.6\n","22:41:41.651 INFO  ProgressMeter -       chr12:24928221             35.1                104610           2977.3\n","22:41:52.888 INFO  ProgressMeter -       chr12:25040970             35.3                105130           2976.2\n","22:42:03.347 INFO  ProgressMeter -       chr12:25177790             35.5                105720           2978.2\n","22:42:15.534 INFO  ProgressMeter -       chr12:25186656             35.7                105760           2962.4\n","22:42:33.396 INFO  ProgressMeter -       chr12:25206844             36.0                105850           2940.4\n","22:42:44.775 INFO  ProgressMeter -       chr12:25228880             36.2                105950           2927.7\n","22:42:57.068 INFO  ProgressMeter -       chr12:25311973             36.4                106300           2920.9\n","22:43:08.079 INFO  ProgressMeter -       chr12:25430591             36.6                106820           2920.4\n","22:43:18.261 INFO  ProgressMeter -       chr12:25581688             36.7                107420           2923.3\n","22:43:28.343 INFO  ProgressMeter -       chr12:25812215             36.9                108360           2935.4\n","22:43:38.375 INFO  ProgressMeter -       chr12:25999588             37.1                109160           2943.8\n","22:43:48.904 INFO  ProgressMeter -       chr12:26301769             37.3                110370           2962.4\n","22:43:59.315 INFO  ProgressMeter -       chr12:26492149             37.4                111140           2969.2\n","22:44:09.954 INFO  ProgressMeter -       chr12:26756921             37.6                112220           2983.9\n","22:44:26.733 INFO  ProgressMeter -       chr12:26780856             37.9                112320           2964.6\n","22:44:37.515 INFO  ProgressMeter -       chr12:26865735             38.1                112680           2960.0\n","22:44:47.525 INFO  ProgressMeter -       chr12:26934183             38.2                112980           2954.9\n","22:44:57.881 INFO  ProgressMeter -       chr12:27031856             38.4                113410           2952.9\n","22:45:09.214 INFO  ProgressMeter -       chr12:27139800             38.6                113850           2949.8\n","22:45:19.980 INFO  ProgressMeter -       chr12:27298403             38.8                114520           2953.4\n","22:45:30.117 INFO  ProgressMeter -       chr12:27457474             38.9                115190           2957.8\n","22:45:40.132 INFO  ProgressMeter -       chr12:27595625             39.1                115760           2959.8\n","22:45:50.382 INFO  ProgressMeter -       chr12:27725663             39.3                116310           2960.9\n","22:46:12.906 INFO  ProgressMeter -       chr12:27874386             39.7                116960           2949.3\n","22:46:23.189 INFO  ProgressMeter -       chr12:27960445             39.8                117330           2945.9\n","22:46:40.541 INFO  ProgressMeter -       chr12:28201290             40.1                118290           2948.6\n","22:46:51.079 INFO  ProgressMeter -       chr12:28500483             40.3                119460           2964.8\n","22:47:01.111 INFO  ProgressMeter -       chr12:28844567             40.5                120840           2986.6\n","22:47:11.369 INFO  ProgressMeter -       chr12:29204668             40.6                122280           3009.5\n","22:47:23.177 INFO  ProgressMeter -       chr12:29394743             40.8                123080           3014.6\n","22:47:33.467 INFO  ProgressMeter -       chr12:29655826             41.0                124140           3027.8\n","22:47:44.049 INFO  ProgressMeter -       chr12:29816645             41.2                124790           3030.6\n","22:47:55.729 INFO  ProgressMeter -       chr12:29936566             41.4                125280           3028.2\n","22:48:06.450 INFO  ProgressMeter -       chr12:30264803             41.5                126610           3047.2\n","22:48:16.554 INFO  ProgressMeter -       chr12:30540322             41.7                127750           3062.2\n","22:48:27.145 INFO  ProgressMeter -       chr12:30720420             41.9                128480           3066.8\n","22:48:37.155 INFO  ProgressMeter -       chr12:30884744             42.1                129180           3071.2\n","22:48:47.713 INFO  ProgressMeter -       chr12:31180852             42.2                130340           3085.9\n","22:48:57.989 INFO  ProgressMeter -       chr12:31337935             42.4                131020           3089.5\n","22:49:08.503 INFO  ProgressMeter -       chr12:31368796             42.6                131180           3080.5\n","22:49:18.545 INFO  ProgressMeter -       chr12:31411317             42.8                131400           3073.6\n","22:49:30.016 INFO  ProgressMeter -       chr12:31503225             42.9                131830           3069.9\n","22:49:41.013 INFO  ProgressMeter -       chr12:31603401             43.1                132280           3067.3\n","22:49:51.342 INFO  ProgressMeter -       chr12:31640171             43.3                132460           3059.3\n","22:50:01.352 INFO  ProgressMeter -       chr12:31687937             43.5                132690           3052.8\n","22:50:11.646 INFO  ProgressMeter -       chr12:31703981             43.6                132770           3042.7\n","22:50:22.670 INFO  ProgressMeter -       chr12:31756648             43.8                133020           3035.6\n","22:50:32.710 INFO  ProgressMeter -       chr12:31800990             44.0                133240           3029.1\n","22:50:42.712 INFO  ProgressMeter -       chr12:31866215             44.2                133560           3024.9\n","22:50:53.976 INFO  ProgressMeter -       chr12:31934116             44.3                133890           3019.5\n","22:51:05.410 INFO  ProgressMeter -       chr12:32018684             44.5                134290           3015.6\n","22:51:17.042 INFO  ProgressMeter -       chr12:32101083             44.7                134700           3011.7\n","22:51:27.082 INFO  ProgressMeter -       chr12:32194890             44.9                135130           3010.0\n","22:51:37.117 INFO  ProgressMeter -       chr12:32265138             45.1                135450           3005.9\n","22:51:47.584 INFO  ProgressMeter -       chr12:32317873             45.2                135700           2999.9\n","22:51:57.662 INFO  ProgressMeter -       chr12:32373745             45.4                135960           2994.5\n","22:52:07.690 INFO  ProgressMeter -       chr12:32427494             45.6                136210           2989.0\n","22:52:18.906 INFO  ProgressMeter -       chr12:32490181             45.8                136510           2983.4\n","22:52:33.055 INFO  ProgressMeter -       chr12:32564991             46.0                136840           2975.2\n","22:52:43.868 INFO  ProgressMeter -       chr12:32670538             46.2                137310           2973.8\n","22:52:54.177 INFO  ProgressMeter -       chr12:32724401             46.3                137570           2968.4\n","22:53:04.312 INFO  ProgressMeter -       chr12:32808735             46.5                137940           2965.6\n","22:53:14.316 INFO  ProgressMeter -       chr12:32862181             46.7                138170           2959.9\n","22:53:27.479 INFO  ProgressMeter -       chr12:32926156             46.9                138460           2952.2\n","22:53:45.296 INFO  ProgressMeter -       chr12:32987651             47.2                138750           2939.8\n","22:53:55.470 INFO  ProgressMeter -       chr12:33133966             47.4                139360           2942.2\n","22:54:06.471 INFO  ProgressMeter -       chr12:33377846             47.5                140370           2952.1\n","22:54:16.504 INFO  ProgressMeter -       chr12:33773554             47.7                141940           2974.6\n","22:54:27.061 INFO  ProgressMeter -       chr12:34111474             47.9                143230           2990.6\n","22:54:37.078 INFO  ProgressMeter -       chr12:34220034             48.1                143700           2990.0\n","22:54:48.195 INFO  ProgressMeter -       chr12:34623305             48.2                145230           3010.2\n","22:54:58.465 INFO  ProgressMeter -       chr12:37428958             48.4                154770           3196.6\n","22:55:09.228 INFO  ProgressMeter -       chr12:37466538             48.6                154950           3188.5\n","22:55:19.648 INFO  ProgressMeter -       chr12:37555572             48.8                155350           3185.4\n","22:55:30.304 INFO  ProgressMeter -       chr12:37577218             48.9                155470           3176.3\n","22:55:49.957 INFO  ProgressMeter -       chr12:37816322             49.3                156450           3175.1\n","22:55:59.984 INFO  ProgressMeter -       chr12:38177318             49.4                157860           3192.8\n","22:56:19.890 INFO  ProgressMeter -       chr12:38331066             49.8                158450           3183.4\n","22:56:30.549 INFO  ProgressMeter -       chr12:38750002             50.0                160110           3205.3\n","22:56:42.656 INFO  ProgressMeter -       chr12:39042949             50.2                161300           3216.2\n","22:56:52.918 INFO  ProgressMeter -       chr12:39243637             50.3                162120           3221.5\n","22:57:03.575 INFO  ProgressMeter -       chr12:39477176             50.5                163080           3229.2\n","22:57:14.489 INFO  ProgressMeter -       chr12:39579840             50.7                163520           3226.3\n","22:57:24.968 INFO  ProgressMeter -       chr12:39838279             50.9                164560           3235.7\n","22:57:34.987 INFO  ProgressMeter -       chr12:40005673             51.0                165220           3238.0\n","22:57:47.298 INFO  ProgressMeter -       chr12:40353812             51.2                166610           3252.2\n","22:57:57.546 INFO  ProgressMeter -       chr12:40700824             51.4                167980           3268.0\n","22:58:08.490 INFO  ProgressMeter -       chr12:41044042             51.6                169350           3283.0\n","22:58:19.146 INFO  ProgressMeter -       chr12:41469902             51.8                171000           3303.6\n","22:58:29.590 INFO  ProgressMeter -       chr12:41776159             51.9                172230           3316.2\n","22:58:40.169 INFO  ProgressMeter -       chr12:41987115             52.1                173110           3321.9\n","22:58:50.186 INFO  ProgressMeter -       chr12:42137323             52.3                173740           3323.4\n","22:59:01.669 INFO  ProgressMeter -       chr12:42246808             52.5                174230           3320.6\n","22:59:12.236 INFO  ProgressMeter -       chr12:42331971             52.6                174610           3316.7\n","22:59:23.535 INFO  ProgressMeter -       chr12:42406781             52.8                174940           3311.1\n","22:59:33.835 INFO  ProgressMeter -       chr12:42512429             53.0                175410           3309.2\n","22:59:44.499 INFO  ProgressMeter -       chr12:42583605             53.2                175730           3304.2\n","22:59:54.522 INFO  ProgressMeter -       chr12:42860942             53.4                176840           3314.7\n","23:00:05.034 INFO  ProgressMeter -       chr12:43260838             53.5                178420           3333.3\n","23:00:15.281 INFO  ProgressMeter -       chr12:43493369             53.7                179360           3340.2\n","23:00:31.667 INFO  ProgressMeter -       chr12:43686002             54.0                180150           3338.0\n","23:00:41.716 INFO  ProgressMeter -       chr12:43774212             54.1                180520           3334.5\n","23:00:52.987 INFO  ProgressMeter -       chr12:43892145             54.3                181020           3332.2\n","23:01:08.900 INFO  ProgressMeter -       chr12:44045877             54.6                181650           3327.5\n","23:01:19.779 INFO  ProgressMeter -       chr12:44173034             54.8                182160           3325.8\n","23:01:30.024 INFO  ProgressMeter -       chr12:44438651             54.9                183210           3334.6\n","23:01:40.393 INFO  ProgressMeter -       chr12:44911362             55.1                185090           3358.2\n","23:01:50.811 INFO  ProgressMeter -       chr12:45125413             55.3                185940           3363.1\n","23:02:01.550 INFO  ProgressMeter -       chr12:45202600             55.5                186260           3358.0\n","23:02:12.653 INFO  ProgressMeter -       chr12:45376785             55.7                186970           3359.6\n","23:02:22.771 INFO  ProgressMeter -       chr12:45619382             55.8                187970           3367.3\n","23:02:33.765 INFO  ProgressMeter -       chr12:45701239             56.0                188320           3362.6\n","23:02:44.272 INFO  ProgressMeter -       chr12:45906008             56.2                189180           3367.4\n","23:03:01.270 INFO  ProgressMeter -       chr12:46000949             56.5                189580           3357.6\n","23:03:11.357 INFO  ProgressMeter -       chr12:46192410             56.6                190400           3362.1\n","23:03:21.866 INFO  ProgressMeter -       chr12:46398815             56.8                191230           3366.3\n","23:03:36.746 INFO  ProgressMeter -       chr12:46729674             57.1                192570           3375.2\n","23:03:46.835 INFO  ProgressMeter -       chr12:46989188             57.2                193610           3383.5\n","23:04:07.581 INFO  ProgressMeter -       chr12:47170170             57.6                194370           3376.3\n","23:04:20.031 INFO  ProgressMeter -       chr12:47246707             57.8                194730           3370.4\n","23:04:30.621 INFO  ProgressMeter -       chr12:47336601             58.0                195150           3367.4\n","23:04:41.072 INFO  ProgressMeter -       chr12:47614465             58.1                196320           3377.5\n","23:04:51.301 INFO  ProgressMeter -       chr12:47781202             58.3                197030           3379.8\n","23:05:01.473 INFO  ProgressMeter -       chr12:48060128             58.5                198240           3390.7\n","23:05:12.084 INFO  ProgressMeter -       chr12:48126287             58.6                198520           3385.2\n","23:05:22.145 INFO  ProgressMeter -       chr12:48305944             58.8                199270           3388.3\n","23:05:32.328 INFO  ProgressMeter -       chr12:48366634             59.0                199520           3382.8\n","23:05:43.205 INFO  ProgressMeter -       chr12:48495178             59.2                200050           3381.4\n","23:05:54.398 INFO  ProgressMeter -       chr12:48521948             59.3                200170           3372.8\n","23:06:04.418 INFO  ProgressMeter -       chr12:48584179             59.5                200450           3368.0\n","23:06:14.511 INFO  ProgressMeter -       chr12:48642149             59.7                200710           3362.9\n"]}],"source":["!/content/gatk-4.4.0.0/gatk Mutect2 \\\n","  -R /content/CTC360/hg38.fa \\\n","  -I /content/CTC360/bam/CTC360_pre.RG.bam \\\n","  -tumor PRE \\\n","  -O /content/CTC360/variants/CTC360_pre.unfiltered.vcf.gz\n"]}],"metadata":{"colab":{"provenance":[],"authorship_tag":"ABX9TyOlMoJSAc0/ckWEtEiVKMhg"},"kernelspec":{"display_name":"Python 3","name":"python3"},"language_info":{"name":"python"}},"nbformat":4,"nbformat_minor":0}